diff --git a/application.js b/application.alt.js
index 4ae7a8b..bde0b42 100644
--- a/application.js
+++ b/application.alt.js
@@ -4153,35 +4153,23 @@ function hideFAQButton(e) {
         mobileDirectBridgeCall('hideFAQButton', e);
 }
 function checkDesign(e) {
-    var t = window.matchMedia('(prefers-color-scheme: dark)').matches;
-    (1 == e ||
-        4 == e ||
-        (((5 == e && t) || (7 == e && t)) && !$('body').hasClass('dark'))) &&
-        ($('body').addClass('dark', 500),
+    (1 != e && 4 != e) || $('body').hasClass('dark') ?
+        1 != e &&
+        4 != e &&
+        $('body').hasClass('dark') &&
+        ($('body').removeClass('dark', 500),
+        'undefined' != typeof mapkit &&
+            (map.colorScheme = mapkit.Map.ColorSchemes.Light))
+    :   ($('body').addClass('dark', 500),
         'undefined' != typeof mapkit &&
             (map.colorScheme = mapkit.Map.ColorSchemes.Dark)),
-        (0 == e ||
-            3 == e ||
-            (((5 == e && !t) || (7 == e && !t)) &&
-                $('body').hasClass('dark'))) &&
-            ($('body').removeClass('dark', 500),
-            'undefined' != typeof mapkit &&
-                (map.colorScheme = mapkit.Map.ColorSchemes.Light)),
         2 == e || (6 == e && !$('body').hasClass('bright')) ?
-            ($('body').hasClass('dark') && $('body').removeClass('dark', 500),
-            $('body').addClass('bright', 500),
+            ($('body').addClass('bright', 500),
             'undefined' != typeof mapkit &&
                 (map.colorScheme = mapkit.Map.ColorSchemes.Light))
-        :   2 != e &&
-            6 != e &&
-            $('body').hasClass('bright') &&
+        :   (2 != e || (6 != e && $('body').hasClass('bright'))) &&
             $('body').removeClass('bright', 500);
 }
-function applyColorScheme(e, t) {
-    1 == t || 4 == t || ((5 == t || 7 == t) && e.matches) ?
-        document.body.classList.add('dark')
-    :   document.body.classList.remove('dark');
-}
 function tellParent(e) {
     1 == mobile_bridge_use ?
         mobileBridgeAdd('tell_main', { jscode: e })
@@ -5162,20 +5150,17 @@ function save_settings(e, t, i) {
             type: 'post',
             data: t,
             success: function (t) {
-                e.html(t), scheduleReloadAfterLightboxClose();
+                e.html(t);
             },
             error: function () {},
         });
 }
 function page_reload() {
     $.ajax({
-        url: window.location.href,
+        url: $(this).attr('href'),
         cache: !1,
         success: function () {
-            (
-                void 0 !== shouldReloadAfterIFrameClose &&
-                shouldReloadAfterIFrameClose
-            ) ?
+            isIframe ?
                 tellParent('location.reload();')
             :   mobileBridgeAdd('reload', {});
         },
@@ -6299,27 +6284,6 @@ function updateTimer(e) {
 function isDateValid(e) {
     return e instanceof Date && !isNaN(e);
 }
-function displayJsAlertInfo(e, t, i) {
-    if ('' === t) return;
-    let n = 'info';
-    switch (e) {
-        case 'notice':
-        case 'success':
-            n = 'success';
-            break;
-        case 'warning':
-            n = 'warning';
-            break;
-        case 'alert':
-        case 'error':
-            n = 'danger';
-            break;
-        default:
-            n = 'info';
-    }
-    const s = `\n  <div class="alert alert-${n} alert-dismissible js-alert" role="alert">\n    <button type="button" class="close" data-dismiss="alert" aria-label="Close">\n        <span aria-hidden="true">&times;</span>\n    </button>\n    <span>\n        ${t}\n    </span>\n  </div>`;
-    i && $('.js-alert').remove(), $('#infoBox').append(s);
-}
 Object.values ||
     (Object.values = function (e) {
         return Object.keys(e).map(function (t) {
@@ -7153,8 +7117,8 @@ Object.values ||
                 elw_police: 'Police MCV',
                 emergency_ambulance: 'NEF oder RTH',
                 emergency_welfare: 'Any Emergency Welfare Vehicle',
-                energy_supply: 'NEA50 (Beliebige HiOrg)',
-                energy_supply_2: 'NEA200 (Beliebige HiOrg)',
+                energy_supply: 'NEA50',
+                energy_supply_2: 'NEA200',
                 fbi_bomb_tech: 'FBI Bomb Technician Vehicle',
                 fbi_drone: 'FBI Surveillance Drone',
                 fbi_investigation_unit: 'FBI Investigation Wagon',
@@ -18951,7 +18915,6 @@ Object.values ||
                 coastal_rescue: 'Coastal Rescue',
                 fbi: 'FBI & DEA',
                 firefighting: 'Hasi\u010dsk\xfd v\u016fz',
-                mountain_rescue: 'Horsk\xfd z\xe1chran\xe1\u0159',
                 police: 'Policie',
                 rescue: 'Sanitka',
                 search_and_rescue: 'Search and Rescue',
@@ -19110,12 +19073,6 @@ Object.values ||
                 mass_casualty_equipment:
                     'Mass Casualty Equipment or Medical Equipment Trailer',
                 mobile_shelter: 'Mobile shelter',
-                mountain_atv: 'Horsk\xe1 \u010dty\u0159kolka',
-                mountain_atv_tractive: 'Horsk\xe1 \u010dty\u0159kolka',
-                mountain_rescue: 'Horsk\xe1 z\xe1chrann\xe1 dod\xe1vka',
-                mountain_rescue_crew: 'Horsk\xe1 z\xe1chrann\xe1 dod\xe1vka',
-                mountain_rescue_dogs: 'sala\u0161nick\xfd pes',
-                mountain_snow: 'Sn\u011b\u017en\xfd sk\xfatr',
                 mtw: 'Transportn\xed t\xfdmov\xfd v\u016fz',
                 municipal_police:
                     'Municipal Police Car or Municipal Police Van',
@@ -19162,9 +19119,6 @@ Object.values ||
                     'Traffic Car, Fast Traffic Car, Traffic Motorcycle and Road Rescue',
                 traffic_patrol: 'Any Traffic Patrol Unit',
                 trailers: {
-                    mountain_atv:
-                        'P\u0159\xedv\u011bs na horskou \u010dty\u0159kolku',
-                    mountain_snow: 'Sn\u011bhov\xfd p\u0159\xedv\u011bs',
                     pump: 'P\u0159\xedv\u011bs pro motorov\xe9 \u010derpadlo',
                 },
                 turntable_ladder: 'AZ nebo AP',
@@ -19379,8 +19333,6 @@ Object.values ||
                 mission_positions: 'Body z\xe1jmu (BZ)',
                 mission_positions_section: 'BZ',
                 missions_section: 'Mise',
-                mountain_missions: 'Horsk\xe1 z\xe1chrann\xe1',
-                mountain_rescue: 'Horsk\xe1 z\xe1chrann\xe1 stanice',
                 police_copter_station_missions:
                     'Z\xe1kladna Leteck\xe9 slu\u017eby P\u010cR',
                 police_depot: 'Police Depot',
@@ -27325,7 +27277,7 @@ Object.values ||
                 technical_aid: 'Technical Aid',
                 tow_trucks: 'Tow Trucks',
                 traffic_police: 'Traffic Police',
-                water_rescue: 'RVL',
+                water_rescue: 'Rajavartiolaitos',
             },
             category: 'Luokka',
             colour: 'V\xe4ri',
@@ -27384,8 +27336,6 @@ Object.values ||
                 battalion_chief_unit: 'Johtoauto',
                 bike_police: 'DB-Bike',
                 boot: 'Alukset (yleinen)',
-                border: 'Mik\xe4 tahansa RVL Partioauto',
-                border_dogs: 'Mik\xe4 koiraper\xe4vaunu',
                 brush_air_command: 'Wildland Lead Planes',
                 brush_truck: 'Wildland fire engine',
                 brush_truck_1: 'Type 3 engine',
@@ -27710,7 +27660,7 @@ Object.values ||
                 buildings_section: 'Asemat',
                 clinic_missions: 'Klinikka',
                 coastal_rescue: 'Rajavartiolaitos',
-                coastal_rescue_missions: 'Meripelastus',
+                coastal_rescue_missions: 'Rajavartiolaitos',
                 dispatch_center_missions: 'H\xe4t\xe4keskus',
                 factory_fire_brigade_missions: 'Factory Fire Brigade',
                 federal_police_missions: 'Federal Police Station',
@@ -27748,8 +27698,6 @@ Object.values ||
                 tow_trucks_missions: 'Tow Trucks',
                 user_buildings: 'Omat rakennukset',
                 user_missions: 'Omat teht\xe4v\xe4t',
-                water_rescue_2_missions: 'Rajavartioasema',
-                water_rescue_school: 'Raja- ja Merivartiokoulu',
                 water_watch: 'Vesipelastus',
                 water_watch_missions: 'Vesipelastus',
             },
@@ -64659,7 +64607,6 @@ const AAO_MULTIPLE_KEYS = [
     'car_large',
 ];
 var lastMissionTimeoutID = null,
-    saved_design_mode = '0',
     hideTimeOut,
     showTimeOut,
     mission_speed;
@@ -64998,7 +64945,6 @@ $(function () {
             ],
             ['elw_police', I18n.t('intervention_order.vehicles.elw_police')],
             ['bike_police', I18n.t('intervention_order.vehicles.bike_police')],
-            ['border', I18n.t('intervention_order.vehicles.border')],
             [
                 'police_car_3',
                 I18n.t('intervention_order.vehicles.police_car_3'),
