diff --git a/application.js b/application.alt.js
index 8178420..0bd232e 100644
--- a/application.js
+++ b/application.alt.js
@@ -1102,7 +1102,7 @@ function building_maps_draw(e) {
         i = 5e3;
     if (
         (void 0 !== e.user_id &&
-            void 0 !== user_id &&
+            'undefined' != typeof user_id &&
             user_id != e.user_id &&
             (i = 1e3),
         isMapKitMap())
@@ -1493,7 +1493,11 @@ function vehicleMarkerAdd(e) {
     }
 }
 function vehicleCreateOnMap(e, t) {
-    if ('true' == t.ioverwrite || void 0 === user_id || t.user_id != user_id)
+    if (
+        'true' == t.ioverwrite ||
+        'undefined' == typeof user_id ||
+        t.user_id != user_id
+    )
         if (void 0 !== buildingVehicleGraphicCachePerId.get(t.b)) {
             var i = buildingVehicleGraphicCachePerId.get(t.b);
             void 0 !== vehicle_graphics_sorted[i] &&
@@ -1521,7 +1525,7 @@ function vehicleCreateOnMap(e, t) {
             map.addAnnotation(e),
             (e.element.className = 'mapkit-marker'),
             e.addEventListener('select', function () {
-                void 0 === user_id ?
+                'undefined' == typeof user_id ?
                     $('#signup_from').effect('highlight', {}, 500)
                 :   lightboxOpen('/vehicles/' + t.id),
                     setTimeout(mapkitDeselectAnnotation, 1e3);
@@ -1531,7 +1535,7 @@ function vehicleCreateOnMap(e, t) {
             )),
             iconMapVehicleGenerate(t.in, t.isr, e),
             e.on('click', function () {
-                void 0 === user_id ?
+                'undefined' == typeof user_id ?
                     $('#signup_from').effect('highlight', {}, 500)
                 :   lightboxOpen('/vehicles/' + t.id);
             })),
@@ -1545,7 +1549,7 @@ function vehicleCreateOnMap(e, t) {
         setInterval(function () {
             vehicleSonderrechte(e);
         }, 1e3)),
-        void 0 === user_id || t.user_id == user_id ?
+        'undefined' == typeof user_id || t.user_id == user_id ?
             (e.title = t.caption)
         :   (e.title = '[' + I18n.t('map.alliance') + '] ' + t.caption),
         t.dd < 0 && (t.dd = 0);
@@ -1559,8 +1563,6 @@ function vehicleCreateOnMap(e, t) {
         (e.polyline = t.polyline),
         (e.current_step = 0),
         (e.timer_steps = n.getTime() - 1e3 * t.dd),
-        (e.last_update = 0),
-        (e.end_date = 1e3 * t.ed),
         (e.vehicle_id = t.id),
         isLeaflet() ?
             e.bindTooltip(
@@ -1582,10 +1584,9 @@ function vehicleCreateOnMap(e, t) {
         e.visible ||
             ((e.visible = !0),
             isLeaflet() && e.setOpacity(1),
-            vehicle_markers_per_id.set(t.id, e),
             vehicle_markers.push(e),
             (s = vehicle_markers.length - 1),
-            registerVehicleAnimate(s, e.end_date));
+            vehicleDriveReal(s));
 }
 function vehicleSonderrechte(e) {
     let t = [];
@@ -1745,146 +1746,6 @@ function vehicleDriveReal(e) {
         }, n);
     } else vehicleArrive(t);
 }
-function vehicleAnimate(e) {
-    var t = vehicle_markers[e];
-    if (t.vehicle_marker_deleted) return !0;
-    var i = new Date();
-    let n = 40;
-    void 0 === t.latitude ||
-        mapIsVisible([t.latitude, t.longitude]) ||
-        (n = 1e3);
-    if (i.getTime() - t.last_update < n) return !0;
-    if (
-        ((t.last_update = i.getTime()),
-        0 == t.timer_steps && (t.timer_steps = i.getTime()),
-        !1 === vehicleStepsAvail(t))
-    )
-        return vehicleArrive(t), void deregisterVehicleAnim(e);
-    if (
-        ((start_lat = routes[t.rh][t.current_step][0]),
-        (start_lng = routes[t.rh][t.current_step][1]),
-        void 0 !== routes[t.rh][t.current_step + 1])
-    ) {
-        var s = 0;
-        do {
-            if (
-                ((end_lat = routes[t.rh][t.current_step + 1][0]),
-                (end_lng = routes[t.rh][t.current_step + 1][1]),
-                (offset_step =
-                    ((i.getTime() - t.timer_steps) /
-                        (1e3 * routes[t.rh][t.current_step][2])) *
-                    100),
-                offset_step >= 100)
-            ) {
-                var o =
-                    i.getTime() -
-                    t.timer_steps -
-                    1e3 * routes[t.rh][t.current_step][2];
-                t.current_step++, (t.timer_steps = i.getTime() - o), (s += 1);
-            }
-        } while (
-            offset_step >= 100 &&
-            void 0 !== routes[t.rh][t.current_step + 1]
-        );
-        if (
-            (('undefined' != typeof route_show && 1 != route_show) ||
-                (void 0 !== t.polyline &&
-                    s > 0 &&
-                    (void 0 === t.delete_step_counter_cache &&
-                        (t.delete_step_counter_cache = 0),
-                    (
-                        void 0 === t.latitude ||
-                        mapIsVisible([t.latitude, t.longitude])
-                    ) ?
-                        (spliceLatLngs(
-                            t.polyline,
-                            s + t.delete_step_counter_cache
-                        ),
-                        (t.delete_step_counter_cache = 0))
-                    :   (t.delete_step_counter_cache =
-                            t.delete_step_counter_cache + s))),
-            void 0 !== routes[t.rh][t.current_step] &&
-                void 0 !== routes[t.rh][t.current_step + 1])
-        )
-            if (
-                ((start_lat = routes[t.rh][t.current_step][0]),
-                (start_lng = routes[t.rh][t.current_step][1]),
-                (end_lat = routes[t.rh][t.current_step + 1][0]),
-                (end_lng = routes[t.rh][t.current_step + 1][1]),
-                (offset_step =
-                    ((i.getTime() - t.timer_steps) /
-                        (1e3 * routes[t.rh][t.current_step][2])) *
-                    100),
-                (diff_jb = end_lat - start_lat),
-                (diff_kb = end_lng - start_lng),
-                (current_lat = start_lat + diff_jb * (offset_step / 100)),
-                (current_lng = start_lng + diff_kb * (offset_step / 100)),
-                (t.latitude = current_lat),
-                (t.longitude = current_lng),
-                !1 === mobile_use_web_map &&
-                    1 == mobile_bridge_use &&
-                    4 == mobile_version)
-            ) {
-                mobileBridgeAdd('vehicle_move', {
-                    id: t.vehicle_id,
-                    title: t.title,
-                    sonderrechte: t.sonderrechte.toString(),
-                    latitude: t.latitude,
-                    longitude: t.longitude,
-                    app_icon_path_normal: t.app_icon_path_normal,
-                    app_icon_path_sonderrechte: t.app_icon_path_sonderrechte,
-                    current_step: t.current_step,
-                    r_id: t.rh,
-                });
-            } else
-                isNaN(current_lat) || isNaN(current_lng),
-                    mapIsVisible([current_lat, current_lng]) &&
-                        (isMapKitMap() ?
-                            (t.coordinate = new mapkit.Coordinate(
-                                current_lat,
-                                current_lng
-                            ))
-                        :   t.setLatLng([current_lat, current_lng])),
-                    mapIsVisible([current_lat, current_lng]) ?
-                        (1 == t.performance_invisible &&
-                            ((t.performance_invisible = !1),
-                            isLeaflet() ? t.addTo(map) : map.addAnnotation(t)),
-                        isLeaflet() &&
-                            (vehicle_label ?
-                                t.openTooltip()
-                            :   t.closeTooltip()))
-                    :   (isLeaflet() ?
-                            (t.closeTooltip(), map.removeLayer(t))
-                        :   0 == t.performance_invisible &&
-                            map.removeAnnotation(t),
-                        (t.performance_invisible = !0)),
-                    (new_position = null);
-    } else t.current_step++;
-}
-function vehicleArrive(e) {
-    void 0 !== e.polyline &&
-        ('undefined' != typeof mapkit ?
-            map.removeOverlay(e.polyline)
-        :   map.removeLayer(e.polyline)),
-        'undefined' != typeof mapkit ?
-            map.removeAnnotation(e)
-        :   map.removeLayer(e),
-        (e.vehicle_marker_deleted = !0),
-        1 == mobile_bridge_use &&
-            4 == mobile_version &&
-            mobileBridgeAdd('vehicle_remove', { id: e.vehicle_id });
-}
-function vehicleArriveById(e) {
-    var t = vehicle_markers[e];
-    t && vehicleArrive(t);
-}
-function vehicleStepsAvail(e) {
-    return (
-        void 0 !== routes[e.rh] &&
-        null != routes[e.rh] &&
-        void 0 !== routes[e.rh][e.current_step]
-    );
-}
 function isMapKitMap() {
     return 'undefined' != typeof mapkit;
 }
@@ -3327,21 +3188,19 @@ function missionTimerStart(e) {
         (e.live_current_value_start = e.live_current_value),
         mission_timers.push({
             mission_id: e.id,
-            timer_id: registerMissionProgressTimer(e.id, 1e3 * e.date_end, e),
+            timer: window.setInterval(function () {
+                missionTimer(e);
+            }, 1e3),
         });
 }
 function missionTimerDelete(e) {
     $.each(mission_timers, function (t, i) {
-        i.mission_id == e &&
-            (i.timer && window.clearInterval(i.timer),
-            i.timer_id && TickManager.deregisterObject(i.timer_id));
+        i.mission_id == e && window.clearInterval(i.timer);
     });
 }
 function missionsTimerDelete(e) {
     $.each(mission_timers, function (t, i) {
-        e.includes(i.mission_id) &&
-            (i.timer && window.clearInterval(i.timer),
-            i.timer_id && TickManager.deregisterObject(i.timer_id));
+        e.includes(i.mission_id) && window.clearInterval(i.timer);
     });
 }
 function patientTimerDelete(e) {
@@ -3355,29 +3214,21 @@ function patientTimerDelete(e) {
     );
 }
 function missionTimer(e) {
-    if (e.live_current_value > e.tv) {
-        if (
-            ((sum_time = e.date_end_calc - e.date_start),
-            (done_time = unix_timestamp() - e.date_start),
-            (percent_done = done_time / sum_time),
-            (percent_todo = 1 - percent_done),
-            (saved_current_value = e.live_current_value_start),
-            (e.live_current_value =
-                Math.ceil(percent_todo * (saved_current_value - e.tv)) + e.tv),
-            e.live_current_value <= 0)
-        )
-            return void missionFinish(e);
+    e.live_current_value > e.tv ?
+        ((sum_time = e.date_end_calc - e.date_start),
+        (done_time = unix_timestamp() - e.date_start),
+        (percent_done = done_time / sum_time),
+        (percent_todo = 1 - percent_done),
+        (saved_current_value = e.live_current_value_start),
+        (e.live_current_value =
+            Math.ceil(percent_todo * (saved_current_value - e.tv)) + e.tv),
         ($('#mission_bar_' + e.id).visible(!0) || Math.random() < 0.3) &&
-            $('#mission_bar_' + e.id).css('width', e.live_current_value + '%');
-    } else
-        (
-            e.live_current_value <= 0 &&
-            e.live_current_water_damage_pump_value <= 0
-        ) ?
-            missionFinish(e)
-        :   $('#mission_bar_striper_' + e.id)
-                .removeClass('progress-striped-inner-active')
-                .removeClass('progress-striped-inner-active-resource-safe');
+            $('#mission_bar_' + e.id).css('width', e.live_current_value + '%'))
+    : e.live_current_value <= 0 && e.live_current_water_damage_pump_value <= 0 ?
+        missionFinish(e)
+    :   $('#mission_bar_striper_' + e.id)
+            .removeClass('progress-striped-inner-active')
+            .removeClass('progress-striped-inner-active-resource-safe');
 }
 function patientBarColor(e) {
     e.target_percent <= 0 ?
@@ -3623,6 +3474,67 @@ function vehicle_image_reload() {
                 );
     });
 }
+function missionCountdown(e, t) {
+    e < 0 ||
+        ($('#mission_countdown_' + t).html(formatTime(e, !1)),
+        (e -= 1),
+        setTimeout(function () {
+            missionCountdown(e, t);
+        }, 1e3));
+}
+function taskCountdown(e, t, i, n) {
+    e >= 0 &&
+        ($('#task_countdown_' + i + '-' + t + '-' + n).html(formatTime(e, !1)),
+        (e -= 1),
+        setTimeout(function () {
+            taskCountdown(e, t, i, n);
+        }, 1e3));
+}
+function vehicleArrivalCountdown(e, t, i) {
+    e < 0 ?
+        $('#vehicle_drive_' + t).html(
+            I18n.t('javascript.arrival') +
+                "... <a href='#' class='btn btn-xs btn-default' onclick='window.location.reload();'>" +
+                I18n.t('javascript.reload') +
+                '</a>'
+        )
+    :   (i > 0 ?
+            $('#vehicle_drive_' + t).html(
+                I18n.t('javascript.start_in') + formatTime(i, !1)
+            )
+        :   $('#vehicle_drive_' + t).html(formatTime(e, !1)),
+        (e -= 1),
+        (i -= 1),
+        setTimeout(function () {
+            vehicleArrivalCountdown(e, t, i);
+        }, 1e3));
+}
+function extensionCountdown(e, t) {
+    e < 0 ||
+        ($('#extension_countdown_' + t).html(formatTime(e, !1)),
+        (e -= 1),
+        setTimeout(function () {
+            extensionCountdown(e, t);
+        }, 1e3));
+}
+function educationCountdown(e, t) {
+    e < 0 ?
+        $('#education_schooling_' + t).html(I18n.t('javascript.few_seconds'))
+    :   ($('#education_schooling_' + t).html(formatTime(e, !1)),
+        (e -= 1),
+        setTimeout(function () {
+            educationCountdown(e, t);
+        }, 1e3));
+}
+function educationCountdownInterval(e) {
+    setupTimerNew({
+        callerId: e,
+        $timer: $(`#${e}`),
+        onTimerEnd: function (e) {
+            e.text(I18n.t('javascript.few_seconds')), location.reload();
+        },
+    });
+}
 function vehicleRefitCountdownInterval(e) {
     setupTimerNew({
         callerId: e,
@@ -3687,6 +3599,25 @@ function dailyRewardsUpdate(e) {
         $('#daily_rewards_li').addClass('daily_bonus_not_taken')
     :   $('#daily_rewards_li').removeClass('daily_bonus_not_taken');
 }
+function updateSaleCountDown() {
+    if (
+        (null != saleTimeout &&
+            (clearTimeout(saleTimeout), (saleTimeout = null)),
+        sale_count_down > Date.now())
+    ) {
+        var e = sale_count_down - Date.now(),
+            t = '% ' + count_down_title + ' ' + getFormattedDuration(e) + ' %';
+        $('#sale_countdown').html(t),
+            $('#sale_countdown_mobile').html(t),
+            (saleTimeout = setTimeout(updateSaleCountDown, 1e3));
+    } else {
+        $('#coins_top').removeClass('saleHighlight');
+        var i = $('#sale_countdown');
+        i && i.remove();
+        var n = $('#sale_countdown_mobile');
+        n && n.remove();
+    }
+}
 function setupSaleTimer(e, t, i) {
     var n = i - t,
         s = i - Date.now();
@@ -3710,12 +3641,23 @@ function updateSaleCountdownTimer(e, t) {
         I18n.t('javascript.time_left', { defaultValue: 'Time left:' }) +
         ' ' +
         getFormattedDuration(t);
-    e.find('.sale_countdown_timer').text(i);
+    e.find('.sale_countdown_timer').html(i);
 }
 function updateSaleCountdownProgressBar(e, t, i) {
     var n = Math.floor((100 / i) * t);
     e.find('.sale_countdown_progress_bar').animate({ width: n + '%' }, 500);
 }
+function getFormattedDuration(e) {
+    var t = Math.floor(e / 864e5),
+        i = Math.floor((e % 864e5) / 36e5),
+        n = Math.floor((e % 36e5) / 6e4),
+        s = Math.floor((e % 6e4) / 1e3),
+        o = '';
+    return (
+        t > 0 && (o = t + I18n.t('javascript.days').charAt(0) + ':'),
+        o + padding(i) + ':' + padding(n) + ':' + padding(s)
+    );
+}
 function coinsUpdate(e) {
     1 == mobile_bridge_use &&
         mobileBridgeAdd('coins', { value: number_format(e) });
@@ -3734,7 +3676,7 @@ function coinsUpdate(e) {
                 })),
             $('#coins_top_mobile .coins-value').html(i),
             $('#coins_top .coins-value').html(t),
-            registerGlobalSaleTimer();
+            updateSaleCountDown();
     } else {
         var n = $('#coins_top .coins-value').text();
         $('.coins-value').html(t),
@@ -3826,6 +3768,19 @@ function formatTime(e, t) {
     } else countdown_message = '00';
     return countdown_message;
 }
+function vehicleArrive(e) {
+    void 0 !== e.polyline &&
+        ('undefined' != typeof mapkit ?
+            map.removeOverlay(e.polyline)
+        :   map.removeLayer(e.polyline)),
+        'undefined' != typeof mapkit ?
+            map.removeAnnotation(e)
+        :   map.removeLayer(e),
+        (e.vehicle_marker_deleted = !0),
+        1 == mobile_bridge_use &&
+            4 == mobile_version &&
+            mobileBridgeAdd('vehicle_remove', { id: e.vehicle_id });
+}
 function rand(e, t) {
     return (
         0 === arguments.length && ((e = 0), (t = 32768)),
@@ -4073,7 +4028,7 @@ function play(e) {
         } catch (e) {}
 }
 function isLoggedIn() {
-    return void 0 !== user_id && !isNaN(user_id);
+    return 'undefined' != typeof user_id && !isNaN(user_id);
 }
 function mobileShow(e) {
     currentMobileTab = e;
@@ -4882,6 +4837,9 @@ function bigMapWindowInfront(e) {
         i > n && ($('.bigMapWindow').css('zIndex', t), (i = t + 1)),
         e.css('zIndex') != i && e.css('zIndex', i + 1);
 }
+function padding(e) {
+    return ('0' + e.toString()).substr(-2);
+}
 function toggleVehicleBuilding(e) {
     var t = 0;
     $('#vehicle_building_' + e).is(':visible') ?
@@ -5179,9 +5137,14 @@ function onAndroidBack() {
 }
 function startProgressBar(e) {
     var t =
-        (100 - ((new Date() - e.startTime) / (e.endTime - e.startTime)) * 100) *
-        (e.missionValue / 100);
-    e.$element.css('width', t + '%');
+            100 -
+            ((new Date() - e.startTime) / (e.endTime - e.startTime)) * 100,
+        i = t * (e.missionValue / 100);
+    e.$element.css('width', i + '%'),
+        t >= 0 &&
+            setTimeout(function () {
+                startProgressBar(e);
+            }, 1e3);
 }
 function showEventInfo(e) {
     return (
@@ -5718,10 +5681,16 @@ function allianceChatBanCountdown(e, t) {
 }
 function startAllianceEventIntervalUpdate(e) {
     const t = new Date(e.end_at);
-    isDateValid(t) &&
-        (Date.now() > t ||
-            (updateAllianceEventInfoBox(e),
-            registerAllianceEventTimer(e.id, e, new Date(e.end_at))));
+    if (!isDateValid(t)) return;
+    if (Date.now() > t) return;
+    const i = `event_timer_${e.id}`;
+    stopTimer(i), updateAllianceEventInfoBox(e);
+    const n = setInterval(function () {
+        Date.now() <= t ?
+            updateAllianceEventInfoBox(e)
+        :   stopAllianceEventIntervalUpdate(i, e);
+    }, 1e3);
+    startedIntervals.set(i, n);
 }
 function updateAllianceEventInfoBox(e) {
     eventRunning = !0;
@@ -5747,7 +5716,7 @@ function getSeasonalEventInfoBoxBody(e) {
         o = 0;
     const a = e.planned_missions,
         r = e.missions_completed_by_alliance;
-    let l = eventCompletedMissions[user_id];
+    let l = eventCompletedMissions[currentUserId];
     l || (l = 0);
     let c = I18n.t('javascript.not_qualified');
     if (l > 0) {
@@ -5764,11 +5733,12 @@ function getSeasonalEventInfoBoxBody(e) {
     );
 }
 function openEventHistoryDetails() {}
-function stopAllianceEventIntervalUpdate(e) {
-    (eventRunning = !1),
+function stopAllianceEventIntervalUpdate(e, t) {
+    stopTimer(e),
+        (eventRunning = !1),
         $('#navbar-inner').removeClass('navbar-inner-event'),
         $('#eventInfo').html(''),
-        e.onTimerEnd && e.onTimerEnd();
+        t.onTimerEnd && t.onTimerEnd();
 }
 function receivedEventPayout(e) {
     const t = `\n  <div class="alert alert-info">\n    <button type="button" class="close" aria-label="Close" onclick="onDismissPayoutInfo();">\n        <span aria-hidden="true">&times;</span>\n    </button>\n    <span>\n        ${I18n.t('javascript.current_reward_for_you', { credits: e.credits, event_currency: e.currency })}\n    </span>\n  </div>`;
@@ -6356,43 +6326,6 @@ function displayJsAlertInfo(e, t, i) {
     const s = `\n  <div class="alert alert-${n} alert-dismissible js-alert" role="alert">\n    <button type="button" class="close" data-dismiss="alert" aria-label="Close">\n        <span aria-hidden="true">&times;</span>\n    </button>\n    <span>\n        ${t}\n    </span>\n  </div>`;
     i && $('.js-alert').remove(), $('#infoBox').append(s);
 }
-function getFormattedDuration(e) {
-    let t = Math.floor(e / 864e5),
-        i = Math.floor((e % 864e5) / 36e5),
-        n = Math.floor((e % 36e5) / 6e4),
-        s = Math.floor((e % 6e4) / 1e3),
-        o = '';
-    return (
-        t > 0 && (o = `${t} ${I18n.t('javascript.days')} `),
-        `${o}${padding(i)}:${padding(n)}:${padding(s)}`
-    );
-}
-function padding(e) {
-    return ('0' + e.toString()).substr(-2);
-}
-function detectDevicePerformance(e) {
-    function t() {
-        i++;
-        const s = performance.now();
-        if (s - n < 1e3) requestAnimationFrame(t);
-        else {
-            const t = i / ((s - n) / 1e3);
-            let o;
-            (o =
-                t >= 50 ? 1e3 / 60
-                : t >= 30 ? 1e3 / 30
-                : 1e3 / 15),
-                vehicleAnimDebug &&
-                    console.log(
-                        `Detected FPS: ${t.toFixed(1)}, using refresh rate: ${o.toFixed(1)}ms`
-                    ),
-                e(o);
-        }
-    }
-    let i = 0,
-        n = performance.now();
-    requestAnimationFrame(t);
-}
 function refreshInputOnSliderChanges(e, t, i) {
     (document.getElementById(e).value = t.value),
         updateSliderScale(t.id, t.value, i);
@@ -6506,332 +6439,6 @@ function hideElementWithEase(e) {
 function showElementWithEase(e) {
     e.style.opacity = '1';
 }
-function timerUpdate(e, t, i) {
-    let n = I18n.t('javascript.few_seconds');
-    i > 0 && (n = getFormattedDuration(i)), t.text(n);
-}
-function registerEducationTimer(e, t, i) {
-    const n = `education_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                educationOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                educationOnTick(e, t.$element, -1), location.reload();
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function educationOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerVehicleTimer(e, t, i) {
-    const n = `vehicle_drive_${e}`,
-        s = $(`#vehicle_drive_${e}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: `#vehicle_drive_${e}`,
-            onTick: ({ id: e, params: t }) => {
-                vehicleArrivalTick(e, t.$element, t.start_date, t.end_date);
-            },
-            onEnd: function ({ id: e, params: t }) {
-                vehicleArrivalTick(e, t.$element, t.start_date, t.end_date);
-            },
-            endDate: i,
-            params: { end_date: i, start_date: t, $element: s },
-        }),
-        n
-    );
-}
-function vehicleArrivalTick(e, t, i, n) {
-    const s = Date.now();
-    let o = n - s,
-        a = i - s;
-    o < 0 ?
-        t.html(
-            I18n.t('javascript.arrival') +
-                "... <a href='#' class='btn btn-xs btn-default' onclick='window.location.reload();'>" +
-                I18n.t('javascript.reload') +
-                '</a>'
-        )
-    : a > 0 ? t.text(I18n.t('javascript.start_in') + getFormattedDuration(a))
-    : t.text(getFormattedDuration(o));
-}
-function registerExtensionTimer(e, t, i) {
-    const n = `extension_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                extensionOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                extensionOnTick(e, t.$element, i), location.reload();
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function extensionOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerTaskTimer(e, t, i) {
-    const n = `task_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                taskProgressOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                taskProgressOnTick(e, t.$element, i);
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function taskProgressOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerMissionCountdownTimer(e, t, i) {
-    const n = `mission_countdown_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                missionCountdownOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                missionCountdownOnTick(e, t.$element, i);
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function missionCountdownOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerMissionProgressTimer(e, t, i) {
-    const n = `mission_progress_${e}`;
-    return (
-        TickManager.registerObject({
-            id: n,
-            onTick: ({ params: e }) => {
-                missionProgressOnTick(e);
-            },
-            onEnd: function ({ params: e }) {
-                missionProgressOnTick(e);
-            },
-            endDate: t,
-            params: i,
-        }),
-        n
-    );
-}
-function missionProgressOnTick(e) {
-    missionTimer(e);
-}
-function registerMissionPumpingTimer(e, t, i) {
-    const n = `mission_pumping_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                missionPumpingOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                missionPumpingOnTick(e, t.$element, i);
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function missionPumpingOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerMissionPumpingProgressTimer(e, t, i) {
-    const n = `mission_pumping_progress_${e}`;
-    return (
-        TickManager.registerObject({
-            id: n,
-            onTick: ({ params: e }) => {
-                missionPumpingProgressOnTick(e);
-            },
-            onEnd: function ({ params: e }) {
-                missionPumpingProgressOnTick(e);
-            },
-            endDate: i,
-            params: t,
-        }),
-        n
-    );
-}
-function missionPumpingProgressOnTick(e) {
-    startProgressBar(e);
-}
-function registerPrisonerTransportationTimer(e, t, i) {
-    const n = `prisoner_transportation_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                missionPrisonerTransportationOnTick(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t, remainingTime: i }) {
-                missionPrisonerTransportationOnTick(e, t.$element, i),
-                    location.reload();
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function missionPrisonerTransportationOnTick(e, t, i) {
-    timerUpdate(e, t, i);
-}
-function registerAllianceEventTimer(e, t, i) {
-    const n = `event_timer_${t.id}`;
-    return (
-        TickManager.registerObject({
-            id: n,
-            onTick: ({ params: e }) => {
-                updateAllianceEventInfoBox(e);
-            },
-            onEnd: function ({ params: e }) {
-                stopAllianceEventIntervalUpdate(e);
-            },
-            endDate: i,
-            params: t,
-        }),
-        n
-    );
-}
-function registerTimedEvent(e, t) {
-    TickManager.registerObject({
-        id: `timed_event_${t.attr('id')}`,
-        elementId: t.attr('id'),
-        onTick: ({ params: e, remainingTime: t }) => {
-            e.$element.text(getFormattedDuration(t));
-        },
-        onEnd: ({ params: e }) => {
-            $(e.eventSelector).remove();
-        },
-        endDate: t.data('end-time'),
-        params: { $element: t, eventSelector: e },
-    });
-}
-function registerPatientTimer() {}
-function registerGlobalSaleTimer() {
-    sale_count_down.getTime() - Date.now() <= 0 ||
-        TickManager.registerObject({
-            id: 'sale_countdown',
-            elementId: 'sale_countdown',
-            onTick: ({ remainingTime: e }) => {
-                globalSaleOnTick(e);
-            },
-            onEnd: ({ remainingTime: e }) => {
-                globalSaleOnTick(e);
-            },
-            endDate: sale_count_down.getTime(),
-        });
-}
-function globalSaleOnTick(e) {
-    let t = $('#sale_countdown'),
-        i = $('#sale_countdown_mobile');
-    if (e > 0) {
-        let n = `% ${count_down_title} ${getFormattedDuration(e)} %`;
-        return t.text(n), void i.text(n);
-    }
-    $('#coins_top').removeClass('saleHighlight'),
-        t && t.remove(),
-        i && i.remove();
-}
-function registerSaleTimer(e, t, i) {
-    const n = `sale_${e.attr('id')}`;
-    let s = i - t;
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: e.attr('id'),
-            onTick: ({ params: e, remainingTime: t }) => {
-                saleOnTick(e, t);
-            },
-            onEnd: ({ params: e }) => {
-                saleEnded(e.$element);
-            },
-            endDate: i.getTime(),
-            params: { $element: e, totalDuration: s },
-        }),
-        n
-    );
-}
-function saleOnTick(e, t) {
-    updateSaleCountdownTimer(e.$element, t),
-        updateSaleCountdownProgressBar(e.$element, t, e.totalDuration);
-}
-function registerComplexBaseTimer(e, t, i) {
-    const n = `complex_base_${e}`,
-        s = $(`#${t}`);
-    return (
-        TickManager.registerObject({
-            id: n,
-            elementId: t,
-            onTick: ({ id: e, params: t, remainingTime: i }) => {
-                timerUpdate(e, t.$element, i);
-            },
-            onEnd: function ({ id: e, params: t }) {
-                timerUpdate(e, t.$element, -1), location.reload();
-            },
-            endDate: i,
-            params: { $element: s },
-        }),
-        n
-    );
-}
-function registerVehicleAnimate(e, t) {
-    VehicleTickManager.registerObject({
-        id: `vehicle_anim_${e}`,
-        endDate: t,
-        onTick: function ({ params: e }) {
-            vehicleAnimate(e.vehicle_marker_id);
-        },
-        onEnd: function ({ params: e }) {
-            vehicleAnimate(e.vehicle_marker_id),
-                vehicleArriveById(e.vehicle_marker_id);
-        },
-        params: { vehicle_marker_id: e },
-    });
-}
-function deregisterVehicleAnim(e) {
-    VehicleTickManager.deregisterObject(`vehicle_anim_${e}`);
-}
 Object.values ||
     (Object.values = function (e) {
         return Object.keys(e).map(function (t) {
@@ -23698,10 +23305,10 @@ Object.values ||
                                 (a.start =
                                     'width' === n || 'height' === n ? 1 : 0)));
         }
-        function V(e, t, i, n, s) {
-            return new V.prototype.init(e, t, i, n, s);
+        function W(e, t, i, n, s) {
+            return new W.prototype.init(e, t, i, n, s);
         }
-        function W(e, t) {
+        function V(e, t) {
             var i,
                 n = { height: e },
                 s = 0;
@@ -24257,7 +23864,7 @@ Object.values ||
                 function i(e, t, i, n) {
                     var s, o, a, r, l, c, u, d, h, p;
                     if (
-                        ((t ? t.ownerDocument || t : W) !== L && N(t),
+                        ((t ? t.ownerDocument || t : V) !== L && N(t),
                         (i = i || []),
                         !e || 'string' != typeof e)
                     )
@@ -24301,7 +23908,7 @@ Object.values ||
                             }
                         if (T.qsa && (!B || !B.test(e))) {
                             if (
-                                ((d = u = V),
+                                ((d = u = W),
                                 (h = t),
                                 (p = 9 === r && e),
                                 1 === r &&
@@ -24349,7 +23956,7 @@ Object.values ||
                     return e;
                 }
                 function o(e) {
-                    return (e[V] = !0), e;
+                    return (e[W] = !0), e;
                 }
                 function a(e) {
                     var t = L.createElement('div');
@@ -24493,7 +24100,7 @@ Object.values ||
                                     for (; (t = t[n]); )
                                         if (1 === t.nodeType || s)
                                             if (
-                                                (l = (c = t[V] || (t[V] = {}))[
+                                                (l = (c = t[W] || (t[W] = {}))[
                                                     n
                                                 ]) &&
                                                 l[0] === u
@@ -24532,8 +24139,8 @@ Object.values ||
                 }
                 function y(e, t, i, n, s, a) {
                     return (
-                        n && !n[V] && (n = y(n)),
-                        s && !s[V] && (s = y(s, a)),
+                        n && !n[W] && (n = y(n)),
+                        s && !s[W] && (s = y(s, a)),
                         o(function (o, a, r, l) {
                             var c,
                                 u,
@@ -24617,7 +24224,7 @@ Object.values ||
                                 (i = E.filter[e[r].type].apply(
                                     null,
                                     e[r].matches
-                                ))[V]
+                                ))[W]
                             ) {
                                 for (
                                     n = ++r;
@@ -24769,8 +24376,8 @@ Object.values ||
                     $,
                     F,
                     H,
-                    V = 'sizzle' + -new Date(),
-                    W = e.document,
+                    W = 'sizzle' + -new Date(),
+                    V = e.document,
                     q = 0,
                     U = 0,
                     Z = s(),
@@ -24886,8 +24493,8 @@ Object.values ||
                         );
                     };
                 try {
-                    se.apply((te = oe.call(W.childNodes)), W.childNodes),
-                        te[W.childNodes.length].nodeType;
+                    se.apply((te = oe.call(V.childNodes)), V.childNodes),
+                        te[V.childNodes.length].nodeType;
                 } catch (e) {
                     se = {
                         apply:
@@ -24913,7 +24520,7 @@ Object.values ||
                 (T = i.support = {}),
                 (N = i.setDocument =
                     function (e) {
-                        var t = e ? e.ownerDocument || e : W;
+                        var t = e ? e.ownerDocument || e : V;
                         return (
                                 t !== L && 9 === t.nodeType && t.documentElement
                             ) ?
@@ -24966,9 +24573,9 @@ Object.values ||
                                 })),
                                 (T.getById = a(function (e) {
                                     return (
-                                        (O.appendChild(e).id = V),
+                                        (O.appendChild(e).id = W),
                                         !t.getElementsByName ||
-                                            !t.getElementsByName(V).length
+                                            !t.getElementsByName(W).length
                                     );
                                 })),
                                 T.getById ?
@@ -25140,8 +24747,8 @@ Object.values ||
                                                                 e
                                                             ) === n)
                                                     ) ?
-                                                        e === t || H(W, e) ? -1
-                                                        : i === t || H(W, i) ? 1
+                                                        e === t || H(V, e) ? -1
+                                                        : i === t || H(V, i) ? 1
                                                         : D ?
                                                             ae.call(D, e) -
                                                             ae.call(D, i)
@@ -25179,8 +24786,8 @@ Object.values ||
                                             for (; r[s] === l[s]; ) s++;
                                             return (
                                                 s ? d(r[s], l[s])
-                                                : r[s] === W ? -1
-                                                : l[s] === W ? 1
+                                                : r[s] === V ? -1
+                                                : l[s] === V ? 1
                                                 : 0
                                             );
                                         }),
@@ -25449,8 +25056,8 @@ Object.values ||
                                                         p =
                                                             (c =
                                                                 (u =
-                                                                    f[V] ||
-                                                                    (f[V] =
+                                                                    f[W] ||
+                                                                    (f[W] =
                                                                         {}))[
                                                                     e
                                                                 ] || [])[0] ===
@@ -25479,7 +25086,7 @@ Object.values ||
                                                         }
                                                 } else if (
                                                     v &&
-                                                    (c = (t[V] || (t[V] = {}))[
+                                                    (c = (t[W] || (t[W] = {}))[
                                                         e
                                                     ]) &&
                                                     c[0] === q
@@ -25500,8 +25107,8 @@ Object.values ||
                                                         :   1 !== d.nodeType) ||
                                                             !++h ||
                                                             (v &&
-                                                                ((d[V] ||
-                                                                    (d[V] =
+                                                                ((d[W] ||
+                                                                    (d[W] =
                                                                         {}))[
                                                                     e
                                                                 ] = [q, h]),
@@ -25522,7 +25129,7 @@ Object.values ||
                                         E.setFilters[e.toLowerCase()] ||
                                         i.error('unsupported pseudo: ' + e);
                                 return (
-                                    s[V] ? s(t)
+                                    s[W] ? s(t)
                                     : s.length > 1 ?
                                         ((n = [e, e, '', t]),
                                         (
@@ -25553,7 +25160,7 @@ Object.values ||
                                 var t = [],
                                     i = [],
                                     n = I(e.replace(me, '$1'));
-                                return n[V] ?
+                                return n[W] ?
                                         o(function (e, t, i, s) {
                                             for (
                                                 var o,
@@ -25727,7 +25334,7 @@ Object.values ||
                             o = G[e + ' '];
                         if (!o) {
                             for (t || (t = _(e)), i = t.length; i--; )
-                                (o = w(t[i]))[V] ? n.push(o) : s.push(o);
+                                (o = w(t[i]))[W] ? n.push(o) : s.push(o);
                             o = G(e, k(s, n));
                         }
                         return o;
@@ -25735,7 +25342,7 @@ Object.values ||
                     (E.pseudos.nth = E.pseudos.eq),
                     (C.prototype = E.filters = E.pseudos),
                     (E.setFilters = new C()),
-                    (T.sortStable = V.split('').sort(J).join('') === V),
+                    (T.sortStable = W.split('').sort(J).join('') === W),
                     N(),
                     [0, 0].sort(J),
                     (T.detectDuplicates = Y),
@@ -27532,8 +27139,8 @@ Object.values ||
                 },
             });
         var He = /^.[^:#\[\.,]*$/,
-            Ve = /^(?:parents|prev(?:Until|All))/,
-            We = ue.expr.match.needsContext,
+            We = /^(?:parents|prev(?:Until|All))/,
+            Ve = ue.expr.match.needsContext,
             qe = { children: !0, contents: !0, next: !0, prev: !0 };
         ue.fn.extend({
             find: function (e) {
@@ -27573,7 +27180,7 @@ Object.values ||
             is: function (e) {
                 return !!h(
                     this,
-                    'string' == typeof e && We.test(e) ? ue(e) : e || [],
+                    'string' == typeof e && Ve.test(e) ? ue(e) : e || [],
                     !1
                 ).length;
             },
@@ -27584,7 +27191,7 @@ Object.values ||
                         s = this.length,
                         o = [],
                         a =
-                            We.test(e) || 'string' != typeof e ?
+                            Ve.test(e) || 'string' != typeof e ?
                                 ue(e, t || this.context)
                             :   0;
                     n < s;
@@ -27678,7 +27285,7 @@ Object.values ||
                             n && 'string' == typeof n && (s = ue.filter(n, s)),
                             this.length > 1 &&
                                 (qe[e] || (s = ue.unique(s)),
-                                Ve.test(e) && (s = s.reverse())),
+                                We.test(e) && (s = s.reverse())),
                             this.pushStack(s)
                         );
                     };
@@ -28674,7 +28281,7 @@ Object.values ||
             $t = ue.fn.load,
             Ft = {},
             Ht = {},
-            Vt = '*/'.concat('*');
+            Wt = '*/'.concat('*');
         try {
             Mt = G.href;
         } catch (e) {
@@ -28750,7 +28357,7 @@ Object.values ||
                     contentType:
                         'application/x-www-form-urlencoded; charset=UTF-8',
                     accepts: {
-                        '*': Vt,
+                        '*': Wt,
                         'text': 'text/plain',
                         'html': 'text/html',
                         'xml': 'application/xml, text/xml',
@@ -28951,7 +28558,7 @@ Object.values ||
                         h.dataTypes[0] && h.accepts[h.dataTypes[0]] ?
                             h.accepts[h.dataTypes[0]] +
                                 ('*' !== h.dataTypes[0] ?
-                                    ', ' + Vt + '; q=0.01'
+                                    ', ' + Wt + '; q=0.01'
                                 :   '')
                         :   h.accepts['*']
                     ),
@@ -29050,12 +28657,12 @@ Object.values ||
                     };
                 }
             });
-        var Wt = [],
+        var Vt = [],
             qt = /(=)\?(?=&|$)|\?\?/;
         ue.ajaxSetup({
             jsonp: 'callback',
             jsonpCallback: function () {
-                var e = Wt.pop() || ue.expando + '_' + Pt++;
+                var e = Vt.pop() || ue.expando + '_' + Pt++;
                 return (this[e] = !0), e;
             },
         }),
@@ -29100,7 +28707,7 @@ Object.values ||
                             (e[o] = a),
                                 i[o] &&
                                     ((i.jsonpCallback = n.jsonpCallback),
-                                    Wt.push(o)),
+                                    Vt.push(o)),
                                 r && ue.isFunction(a) && a(r[0]),
                                 (r = a = t);
                         }),
@@ -29262,9 +28869,9 @@ Object.values ||
                 t ? ti.unshift(e) : ti.push(e);
             },
         })),
-            (ue.Tween = V),
-            (V.prototype = {
-                constructor: V,
+            (ue.Tween = W),
+            (W.prototype = {
+                constructor: W,
                 init: function (e, t, i, n, s, o) {
                     (this.elem = e),
                         (this.prop = i),
@@ -29275,14 +28882,14 @@ Object.values ||
                         (this.unit = o || (ue.cssNumber[i] ? '' : 'px'));
                 },
                 cur: function () {
-                    var e = V.propHooks[this.prop];
+                    var e = W.propHooks[this.prop];
                     return e && e.get ?
                             e.get(this)
-                        :   V.propHooks._default.get(this);
+                        :   W.propHooks._default.get(this);
                 },
                 run: function (e) {
                     var t,
-                        i = V.propHooks[this.prop];
+                        i = W.propHooks[this.prop];
                     return (
                         this.options.duration ?
                             (this.pos = t =
@@ -29299,13 +28906,13 @@ Object.values ||
                             this.options.step.call(this.elem, this.now, this),
                         i && i.set ?
                             i.set(this)
-                        :   V.propHooks._default.set(this),
+                        :   W.propHooks._default.set(this),
                         this
                     );
                 },
             }),
-            (V.prototype.init.prototype = V.prototype),
-            (V.propHooks = {
+            (W.prototype.init.prototype = W.prototype),
+            (W.propHooks = {
                 _default: {
                     get: function (e) {
                         var t;
@@ -29336,7 +28943,7 @@ Object.values ||
                     },
                 },
             }),
-            (V.propHooks.scrollTop = V.propHooks.scrollLeft =
+            (W.propHooks.scrollTop = W.propHooks.scrollLeft =
                 {
                     set: function (e) {
                         e.elem.nodeType &&
@@ -29349,7 +28956,7 @@ Object.values ||
                 ue.fn[t] = function (e, n, s) {
                     return null == e || 'boolean' == typeof e ?
                             i.apply(this, arguments)
-                        :   this.animate(W(t, !0), e, n, s);
+                        :   this.animate(V(t, !0), e, n, s);
                 };
             }),
             ue.fn.extend({
@@ -29437,9 +29044,9 @@ Object.values ||
             }),
             ue.each(
                 {
-                    slideDown: W('show'),
-                    slideUp: W('hide'),
-                    slideToggle: W('toggle'),
+                    slideDown: V('show'),
+                    slideUp: V('hide'),
+                    slideToggle: V('toggle'),
                     fadeIn: { opacity: 'show' },
                     fadeOut: { opacity: 'hide' },
                     fadeToggle: { opacity: 'toggle' },
@@ -29483,7 +29090,7 @@ Object.values ||
                 },
             }),
             (ue.timers = []),
-            (ue.fx = V.prototype.init),
+            (ue.fx = W.prototype.init),
             (ue.fx.tick = function () {
                 var e,
                     i = ue.timers,
@@ -38422,8 +38029,8 @@ Object.values ||
                             new Date($.getFullYear(), $.getMonth(), $.getDate())
                         ),
                         H = this._get(e, 'isRTL'),
-                        V = this._get(e, 'showButtonPanel'),
-                        W = this._get(e, 'hideIfNoPrevNext'),
+                        W = this._get(e, 'showButtonPanel'),
+                        V = this._get(e, 'hideIfNoPrevNext'),
                         q = this._get(e, 'navigationAsDateFormat'),
                         U = this._getNumberOfMonths(e),
                         Z = this._get(e, 'showCurrentAtPos'),
@@ -38479,7 +38086,7 @@ Object.values ||
                                     "'>" +
                                     i +
                                     '</span></a>'
-                                : W ? ''
+                                : V ? ''
                                 : "<a class='ui-datepicker-prev ui-corner-all ui-state-disabled' title='" +
                                     i +
                                     "'><span class='ui-icon ui-icon-circle-triangle-" +
@@ -38507,7 +38114,7 @@ Object.values ||
                                     "'>" +
                                     s +
                                     '</span></a>'
-                                : W ? ''
+                                : V ? ''
                                 : "<a class='ui-datepicker-next ui-corner-all ui-state-disabled' title='" +
                                     s +
                                     "'><span class='ui-icon ui-icon-circle-triangle-" +
@@ -38535,7 +38142,7 @@ Object.values ||
                                     '</button>'
                                 ),
                             c =
-                                V ?
+                                W ?
                                     "<div class='ui-datepicker-buttonpane ui-widget-content'>" +
                                     (H ? l : '') +
                                     (this._isInRange(e, r) ?
@@ -49221,7 +48828,7 @@ Object.values ||
                 e.addEventListener(ii, n, !1),
                 e.addEventListener(ni, n, !1);
         }
-        function V(e, t, i) {
+        function W(e, t, i) {
             function n(e) {
                 var t;
                 if (qt) {
@@ -49264,7 +48871,7 @@ Object.values ||
                 this
             );
         }
-        function W(e, t) {
+        function V(e, t) {
             var i = e[ui + li + t],
                 n = e[ui + ci + t],
                 s = e[ui + 'dblclick' + t];
@@ -49447,7 +49054,7 @@ Object.values ||
                 },
                 r = a;
             qt && 0 === t.indexOf('touch') ? j(e, t, a, o)
-            : !Ut || 'dblclick' !== t || !V || (qt && Pt) ?
+            : !Ut || 'dblclick' !== t || !W || (qt && Pt) ?
                 'addEventListener' in e ?
                     'mousewheel' === t ?
                         e.addEventListener(
@@ -49471,7 +49078,7 @@ Object.values ||
                             }),
                         e.addEventListener(t, a, !1))
                 :   'attachEvent' in e && e.attachEvent('on' + t, a)
-            :   V(e, a, o),
+            :   W(e, a, o),
                 (e[bi] = e[bi] || {}),
                 (e[bi][o] = a);
         }
@@ -49480,7 +49087,7 @@ Object.values ||
                 a = e[bi] && e[bi][o];
             if (!a) return this;
             qt && 0 === t.indexOf('touch') ? D(e, t, o)
-            : !Ut || 'dblclick' !== t || !W || (qt && Pt) ?
+            : !Ut || 'dblclick' !== t || !V || (qt && Pt) ?
                 'removeEventListener' in e ?
                     'mousewheel' === t ?
                         e.removeEventListener(
@@ -49496,7 +49103,7 @@ Object.values ||
                             !1
                         )
                 :   'detachEvent' in e && e.detachEvent('on' + t, a)
-            :   W(e, o),
+            :   V(e, o),
                 (e[bi][o] = null);
         }
         function be(e) {
@@ -49714,7 +49321,7 @@ Object.values ||
             }
             return e;
         }
-        function Ve(e, t) {
+        function We(e, t) {
             var i,
                 n,
                 s,
@@ -49723,7 +49330,7 @@ Object.values ||
                 r = a ? a.coordinates : null,
                 l = [],
                 c = t && t.pointToLayer,
-                u = (t && t.coordsToLatLng) || We;
+                u = (t && t.coordsToLatLng) || Ve;
             if (!r && !a) return null;
             switch (a.type) {
                 case 'Point':
@@ -49746,7 +49353,7 @@ Object.values ||
                     );
                 case 'GeometryCollection':
                     for (s = 0, o = a.geometries.length; s < o; s++) {
-                        var d = Ve(
+                        var d = We(
                             {
                                 geometry: a.geometries[s],
                                 type: 'Feature',
@@ -49761,12 +49368,12 @@ Object.values ||
                     throw new Error('Invalid GeoJSON object.');
             }
         }
-        function We(e) {
+        function Ve(e) {
             return new S(e[1], e[0], e[2]);
         }
         function qe(e, t, i) {
             for (var n, s = [], o = 0, a = e.length; o < a; o++)
-                (n = t ? qe(e[o], t - 1, i) : (i || We)(e[o])), s.push(n);
+                (n = t ? qe(e[o], t - 1, i) : (i || Ve)(e[o])), s.push(n);
             return s;
         }
         function Ue(e, t) {
@@ -50515,9 +50122,9 @@ Object.values ||
             $t = !window.L_DISABLE_3D && (Ot || Rt || Bt) && !Nt && !Dt,
             Ft = 'undefined' != typeof orientation || I('mobile'),
             Ht = Ft && Ct,
-            Vt = Ft && Rt,
-            Wt = !window.PointerEvent && window.MSPointerEvent,
-            qt = !(!window.PointerEvent && !Wt),
+            Wt = Ft && Rt,
+            Vt = !window.PointerEvent && window.MSPointerEvent,
+            qt = !(!window.PointerEvent && !Vt),
             Ut =
                 !window.L_NO_TOUCH &&
                 (qt ||
@@ -50567,8 +50174,8 @@ Object.values ||
                 any3d: $t,
                 mobile: Ft,
                 mobileWebkit: Ht,
-                mobileWebkit3d: Vt,
-                msPointer: Wt,
+                mobileWebkit3d: Wt,
+                msPointer: Vt,
                 pointer: qt,
                 touch: Ut,
                 mobileOpera: Zt,
@@ -50578,20 +50185,20 @@ Object.values ||
                 svg: Jt,
                 vml: Qt,
             }),
-            ei = Wt ? 'MSPointerDown' : 'pointerdown',
-            ti = Wt ? 'MSPointerMove' : 'pointermove',
-            ii = Wt ? 'MSPointerUp' : 'pointerup',
-            ni = Wt ? 'MSPointerCancel' : 'pointercancel',
+            ei = Vt ? 'MSPointerDown' : 'pointerdown',
+            ti = Vt ? 'MSPointerMove' : 'pointermove',
+            ii = Vt ? 'MSPointerUp' : 'pointerup',
+            ni = Vt ? 'MSPointerCancel' : 'pointercancel',
             si = ['INPUT', 'SELECT', 'OPTION'],
             oi = {},
             ai = !1,
             ri = 0,
             li =
-                Wt ? 'MSPointerDown'
+                Vt ? 'MSPointerDown'
                 : qt ? 'pointerdown'
                 : 'touchstart',
             ci =
-                Wt ? 'MSPointerUp'
+                Vt ? 'MSPointerUp'
                 : qt ? 'pointerup'
                 : 'touchend',
             ui = '_leaflet_',
@@ -52643,7 +52250,7 @@ Object.values ||
                 Mercator: Fi,
                 SphericalMercator: _t,
             }),
-            Vi = t({}, mt, {
+            Wi = t({}, mt, {
                 code: 'EPSG:3395',
                 projection: Fi,
                 transformation: (function () {
@@ -52651,7 +52258,7 @@ Object.values ||
                     return E(e, 0.5, -e, 0.5);
                 })(),
             }),
-            Wi = t({}, mt, {
+            Vi = t({}, mt, {
                 code: 'EPSG:4326',
                 projection: $i,
                 transformation: E(1 / 180, 1, -1 / 180, 0.5),
@@ -52673,10 +52280,10 @@ Object.values ||
                 infinite: !0,
             });
         (pt.Earth = mt),
-            (pt.EPSG3395 = Vi),
+            (pt.EPSG3395 = Wi),
             (pt.EPSG3857 = bt),
             (pt.EPSG900913 = yt),
-            (pt.EPSG4326 = Wi),
+            (pt.EPSG4326 = Vi),
             (pt.Simple = qi);
         var Ui = dt.extend({
             options: {
@@ -53803,7 +53410,7 @@ Object.values ||
                     }
                     var o = this.options;
                     if (o.filter && !o.filter(e)) return this;
-                    var a = Ve(e, o);
+                    var a = We(e, o);
                     return a ?
                             ((a.feature = Ge(e)),
                             (a.defaultOptions = a.options),
@@ -55449,7 +55056,7 @@ Object.values ||
                         s = n.min,
                         o = n.max,
                         a = (
-                            this._wmsVersion >= 1.3 && this._crs === Wi ?
+                            this._wmsVersion >= 1.3 && this._crs === Vi ?
                                 [s.y, s.x, o.y, o.x]
                             :   [s.x, s.y, o.x, o.y]).join(','),
                         r = _n.prototype.getTileUrl.call(this, e);
@@ -56121,8 +55728,8 @@ Object.values ||
         });
         (kn.create = wn),
             (kn.pointsToPath = P),
-            (on.geometryToLayer = Ve),
-            (on.coordsToLatLng = We),
+            (on.geometryToLayer = We),
+            (on.coordsToLatLng = Ve),
             (on.coordsToLatLngs = qe),
             (on.latLngToCoords = Ue),
             (on.latLngsToCoords = Ze),
@@ -57629,7 +57236,6 @@ Object.values ||
     })(this);
 const isSafari = void 0 !== window.safari;
 var currentUserId = -1,
-    user_id = -1,
     map,
     alliance_member_buildings_show,
     geocoder,
@@ -57671,7 +57277,6 @@ var currentUserId = -1,
     mixed_mobile_desktop_mode = !1,
     mobile_version = 1,
     vehicle_markers = Array(),
-    vehicle_markers_per_id = new Map(),
     icon_empty,
     vehicles_not_involved = Array(),
     alliance_show_not_involved_vehicle = !1,
@@ -57771,26 +57376,6 @@ var lastMissionTimeoutID = null,
 let client_uses_design_v2 = !0,
     magicValueAvailableSpace = 71;
 const startedIntervals = new Map();
-var pauseTick = !1,
-    pauseVehicleAnim = !1,
-    tickDebug = !1,
-    vehicleAnimDebug = !1,
-    requestAnimationFrame =
-        window.requestAnimationFrame ||
-        window.webkitRequestAnimationFrame ||
-        window.mozRequestAnimationFrame ||
-        window.oRequestAnimationFrame ||
-        window.msRequestAnimationFrame,
-    cancelAnimationFrame =
-        window.cancelAnimationFrame ||
-        window.webkitCancelRequestAnimationFrame ||
-        window.webkitCancelAnimationFrame ||
-        window.mozCancelRequestAnimationFrame ||
-        window.mozCancelAnimationFrame ||
-        window.oCancelRequestAnimationFrame ||
-        window.oCancelAnimationFrame ||
-        window.msCancelRequestAnimationFrame ||
-        window.msCancelAnimationFrame;
 $(function () {
     function onCoinsTop() {
         return !mobile_bridge_use || (mobileBridgeAdd('coins_window', {}), !1);
@@ -63549,10 +63134,10 @@ if (
                 : ye.indexOf('|' + i + '|') < 0
             );
         }
-        function V(e, t) {
+        function W(e, t) {
             t.style.cssText = e.style.cssText + t.style.cssText;
         }
-        function W(e) {
+        function V(e) {
             var t = function (e) {
                 for (; H(e.parentNode, !0); ) e = e.parentNode;
                 return e;
@@ -63564,7 +63149,7 @@ if (
                     var s = t(e),
                         o = K(s, e),
                         a = e;
-                    V(s, a), C(o, s), C(a, s);
+                    W(s, a), C(o, s), C(a, s);
                 }
                 if (n && k(e, i) && k(e.parentNode, i)) {
                     var l = z(e, 'li');
@@ -64159,7 +63744,7 @@ if (
                 N,
                 L,
                 R,
-                V,
+                W,
                 q,
                 U,
                 K,
@@ -64186,8 +63771,8 @@ if (
                 $e,
                 Fe,
                 He,
-                Ve,
                 We,
+                Ve,
                 qe,
                 Ue,
                 Ze,
@@ -64236,7 +63821,7 @@ if (
                     var t = le.formats[kt.format];
                     'init' in (i = t ? new t() : {}) && i.init.call(_t),
                         Ie(),
-                        Ve(),
+                        We(),
                         Be(),
                         Pe(),
                         $e(),
@@ -64338,7 +63923,7 @@ if (
                         p(z, 'keyup', nt),
                         p(z, 'blur', dt),
                         p(z, 'keyup', ht),
-                        p(z, 'paste', We),
+                        p(z, 'paste', Ve),
                         p(z, i, Je),
                         p(z, s, st),
                         p(z, n, Qe),
@@ -64521,7 +64106,7 @@ if (
                                 e.preventDefault();
                         });
                 }),
-                (Ve = function () {
+                (We = function () {
                     var e = kt.emoticons,
                         t = kt.emoticonsRoot || '';
                     e && (wt = n({}, e.more, e.dropdown, e.hidden)),
@@ -64718,7 +64303,7 @@ if (
                         !e.defaultPrevented &&
                         (pt(), _t.closeDropDown());
                 }),
-                (We = function (e) {
+                (Ve = function (e) {
                     var t = Le || ze,
                         i = z,
                         n = e.clipboardData,
@@ -64769,7 +64354,7 @@ if (
                     Y.call('pasteRaw', e),
                         j(a, 'pasteraw', e),
                         e.html ?
-                            ((t.innerHTML = e.html), W(t))
+                            ((t.innerHTML = e.html), V(t))
                         :   (t.innerHTML = X(e.text || ''));
                     var n = { val: t.innerHTML };
                     'fragmentToSource' in i &&
@@ -64889,7 +64474,7 @@ if (
                         d(n, s[o].cloneNode(!0));
                     return (
                         d(z, n),
-                        W(n),
+                        V(n),
                         u(n),
                         (t = n.innerHTML),
                         !1 !== e &&
@@ -65450,7 +65035,7 @@ if (
                         var t,
                             i = _t.sourceMode(),
                             n = !i && K.hasSelection();
-                        (V = !1),
+                        (W = !1),
                             (e =
                                 !1 !== e &&
                                 !S.getElementById('sceditor-start-marker')),
@@ -65473,7 +65058,7 @@ if (
                         n = 13 === i || 32 === i,
                         s = 8 === i || 46 === i;
                     (ht.lastChar = t),
-                        V ||
+                        W ||
                             (13 === t || 32 === t ?
                                 n ? (ht.triggerNext = !0)
                                 :   ut()
@@ -65483,11 +65068,11 @@ if (
                             :   ht.triggerNext && (ut(), (ht.triggerNext = !1)),
                             clearTimeout(q),
                             (q = setTimeout(function () {
-                                V || ut();
+                                W || ut();
                             }, 1500)));
                 }),
                 (Je = function (e) {
-                    (V = /start/i.test(e.type)) || ut();
+                    (W = /start/i.test(e.type)) || ut();
                 }),
                 (pt = function () {
                     _t.updateOriginal();
@@ -66410,8 +65995,8 @@ if (
                     blockLevelList: ye,
                     canHaveChildren: F,
                     isInline: H,
-                    copyCSS: V,
-                    fixNesting: W,
+                    copyCSS: W,
+                    fixNesting: V,
                     findCommonAncestor: q,
                     getSibling: U,
                     removeWhiteSpace: Z,
@@ -66757,10 +66342,10 @@ if (
                 : ye.indexOf('|' + i + '|') < 0
             );
         }
-        function V(e, t) {
+        function W(e, t) {
             t.style.cssText = e.style.cssText + t.style.cssText;
         }
-        function W(e) {
+        function V(e) {
             var t = function (e) {
                 for (; H(e.parentNode, !0); ) e = e.parentNode;
                 return e;
@@ -66772,7 +66357,7 @@ if (
                     var s = t(e),
                         o = K(s, e),
                         a = e;
-                    V(s, a), C(o, s), C(a, s);
+                    W(s, a), C(o, s), C(a, s);
                 }
                 if (n && k(e, i) && k(e.parentNode, i)) {
                     var l = z(e, 'li');
@@ -67367,7 +66952,7 @@ if (
                 N,
                 L,
                 R,
-                V,
+                W,
                 q,
                 U,
                 K,
@@ -67394,8 +66979,8 @@ if (
                 $e,
                 Fe,
                 He,
-                Ve,
                 We,
+                Ve,
                 qe,
                 Ue,
                 Ze,
@@ -67444,7 +67029,7 @@ if (
                     var t = le.formats[kt.format];
                     'init' in (i = t ? new t() : {}) && i.init.call(_t),
                         Ie(),
-                        Ve(),
+                        We(),
                         Be(),
                         Pe(),
                         $e(),
@@ -67546,7 +67131,7 @@ if (
                         p(z, 'keyup', nt),
                         p(z, 'blur', dt),
                         p(z, 'keyup', ht),
-                        p(z, 'paste', We),
+                        p(z, 'paste', Ve),
                         p(z, i, Je),
                         p(z, s, st),
                         p(z, n, Qe),
@@ -67729,7 +67314,7 @@ if (
                                 e.preventDefault();
                         });
                 }),
-                (Ve = function () {
+                (We = function () {
                     var e = kt.emoticons,
                         t = kt.emoticonsRoot || '';
                     e && (wt = n({}, e.more, e.dropdown, e.hidden)),
@@ -67926,7 +67511,7 @@ if (
                         !e.defaultPrevented &&
                         (pt(), _t.closeDropDown());
                 }),
-                (We = function (e) {
+                (Ve = function (e) {
                     var t = Le || ze,
                         i = z,
                         n = e.clipboardData,
@@ -67977,7 +67562,7 @@ if (
                     Y.call('pasteRaw', e),
                         j(a, 'pasteraw', e),
                         e.html ?
-                            ((t.innerHTML = e.html), W(t))
+                            ((t.innerHTML = e.html), V(t))
                         :   (t.innerHTML = X(e.text || ''));
                     var n = { val: t.innerHTML };
                     'fragmentToSource' in i &&
@@ -68097,7 +67682,7 @@ if (
                         d(n, s[o].cloneNode(!0));
                     return (
                         d(z, n),
-                        W(n),
+                        V(n),
                         u(n),
                         (t = n.innerHTML),
                         !1 !== e &&
@@ -68658,7 +68243,7 @@ if (
                         var t,
                             i = _t.sourceMode(),
                             n = !i && K.hasSelection();
-                        (V = !1),
+                        (W = !1),
                             (e =
                                 !1 !== e &&
                                 !S.getElementById('sceditor-start-marker')),
@@ -68681,7 +68266,7 @@ if (
                         n = 13 === i || 32 === i,
                         s = 8 === i || 46 === i;
                     (ht.lastChar = t),
-                        V ||
+                        W ||
                             (13 === t || 32 === t ?
                                 n ? (ht.triggerNext = !0)
                                 :   ut()
@@ -68691,11 +68276,11 @@ if (
                             :   ht.triggerNext && (ut(), (ht.triggerNext = !1)),
                             clearTimeout(q),
                             (q = setTimeout(function () {
-                                V || ut();
+                                W || ut();
                             }, 1500)));
                 }),
                 (Je = function (e) {
-                    (V = /start/i.test(e.type)) || ut();
+                    (W = /start/i.test(e.type)) || ut();
                 }),
                 (pt = function () {
                     _t.updateOriginal();
@@ -69618,8 +69203,8 @@ if (
                     blockLevelList: ye,
                     canHaveChildren: F,
                     isInline: H,
-                    copyCSS: V,
-                    fixNesting: W,
+                    copyCSS: W,
+                    fixNesting: V,
                     findCommonAncestor: q,
                     getSibling: U,
                     removeWhiteSpace: Z,
@@ -75533,173 +75118,3 @@ var show_only_available = !0,
     sea_mission_alliance_rescue_txt = '',
     alliance_cells_txt = '',
     vehicle_data = new Map();
-const TickManager = (() => {
-        function e(e) {
-            tickDebug && console.log(e);
-        }
-        function t() {
-            c ||
-                (e('startInterval'),
-                (c = setInterval(function () {
-                    requestAnimationFrame(n);
-                }, d)));
-        }
-        function i() {
-            c &&
-                0 === l.size &&
-                (e('stopInterval'), clearInterval(c), (c = null));
-        }
-        function n() {
-            e(`tickAll: ${l.size} objects`);
-            const t = Date.now();
-            if (null !== u) {
-                e(`delta: ${t - u}`);
-            }
-            if (((u = t), pauseTick)) e(`ticks are paused. skipping at ${t}`);
-            else {
-                for (const [e, i] of l.entries())
-                    try {
-                        if (i.endDate) {
-                            const n = i.endDate - t;
-                            if (n <= 0) {
-                                a(i), l.delete(e);
-                                continue;
-                            }
-                            i.remainingTime = n;
-                        }
-                        o(i);
-                    } catch (e) {
-                        console.error(e);
-                    }
-                i();
-            }
-        }
-        function s({
-            id: i,
-            elementId: n,
-            onTick: s,
-            onEnd: a,
-            endDate: r,
-            params: c,
-        }) {
-            if (!i || 'function' != typeof s)
-                return void e(`Invalid object registration: ${i}`);
-            if (!r)
-                return void e(`Invalid or missing endDate for object: ${i}`);
-            r instanceof Date && (r = r.getTime());
-            const u = {
-                id: i,
-                elementId: n,
-                onTick: s,
-                onEnd: a,
-                endDate: r,
-                params: c,
-            };
-            (u.remainingTime = u.endDate - Date.now()), l.set(i, u), o(u), t();
-        }
-        function o(t) {
-            'function' == typeof t.onTick &&
-                (e(`onTick: ${t.id} remainingTime: ${t.remainingTime}`),
-                t.onTick(t));
-        }
-        function a(t) {
-            'function' == typeof t.onEnd &&
-                (e(`onEnd: ${t.id} remainingTime: ${t.remainingTime}`),
-                t.onEnd(t));
-        }
-        function r(e) {
-            l.delete(e), i();
-        }
-        const l = new Map();
-        let c = null,
-            u = null;
-        const d = 1e3;
-        return { registerObject: s, deregisterObject: r };
-    })(),
-    VehicleTickManager = (() => {
-        function e(e) {
-            vehicleAnimDebug && console.log(e);
-        }
-        function t() {
-            u ||
-                (e('startVehicleInterval'),
-                (u = setInterval(function () {
-                    requestAnimationFrame(n);
-                }, h)));
-        }
-        function i() {
-            u &&
-                0 === c.size &&
-                (e('stopVehicleInterval'), clearInterval(u), (u = null));
-        }
-        function n() {
-            e(`tickAll: ${c.size} objects`);
-            const t = Date.now();
-            if (null !== d) {
-                e(`delta: ${t - d}`);
-            }
-            if (((d = t), pauseVehicleAnim))
-                e(`ticks are paused. skipping at ${t}`);
-            else {
-                for (const [e, i] of c.entries())
-                    try {
-                        if (i.endDate) {
-                            const n = i.endDate - t;
-                            if (n <= 0) {
-                                a(i), c.delete(e);
-                                continue;
-                            }
-                            i.remainingTime = n;
-                        }
-                        o(i);
-                    } catch (e) {
-                        console.error(e);
-                    }
-                i();
-            }
-        }
-        function s({
-            id: i,
-            elementId: n,
-            onTick: s,
-            onEnd: a,
-            endDate: r,
-            params: l,
-        }) {
-            if (!i || 'function' != typeof s)
-                return void e(`Invalid object registration: ${i}`);
-            if (!r)
-                return void e(`Invalid or missing endDate for object: ${i}`);
-            r instanceof Date && (r = r.getTime());
-            const u = {
-                id: i,
-                elementId: n,
-                onTick: s,
-                onEnd: a,
-                endDate: r,
-                params: l,
-            };
-            (u.remainingTime = u.endDate - Date.now()), c.set(i, u), o(u), t();
-        }
-        function o(t) {
-            'function' == typeof t.onTick &&
-                (e(`onTick: ${t.id} remainingTime: ${t.remainingTime}`),
-                t.onTick(t));
-        }
-        function a(t) {
-            'function' == typeof t.onEnd &&
-                (e(`onEnd: ${t.id} remainingTime: ${t.remainingTime}`),
-                t.onEnd(t));
-        }
-        function r(e) {
-            c.delete(e), i();
-        }
-        function l(e) {
-            h = e;
-        }
-        const c = new Map();
-        let u = null,
-            d = null,
-            h = 1e3 / 24;
-        return { registerObject: s, deregisterObject: r, setRefreshRate: l };
-    })();
