diff --git a/application.js b/application.alt.js
index 384bbf8..1fcce40 100644
--- a/application.js
+++ b/application.alt.js
@@ -1641,6 +1641,10 @@ function prepareBuildingDomElementStr(e, t) {
         0 == e.show_vehicles_at_startpage && (n += "style='display: none' "),
         t && (s = t.map(e => prepareVehicleMarkerStr(e)).join('')),
         (n = n + '>' + s + '</ul></li>'),
+        e.building_type == BUILDING_TYPE_LEITSTELLE &&
+            ((leitstelle_latitude = e.latitude),
+            (leitstelle_longitude = e.longitude),
+            leitstelles.push([e.latitude, e.longitude])),
         n
     );
 }
@@ -7615,61 +7619,6 @@ function initVehiclesEquipment(e, t = {}) {
         i(e.state.vehicleId);
     });
 }
-function loadNextPage(e, t, i, n, s = 200) {
-    if (!1 === additionalLoadAllowed) return void hideLoadingIfPresent();
-    if (activeTab !== e) return void hideLoadingIfPresent();
-    if (0 === $(`#${e}`).find('.expandabletable').length)
-        return void hideLoadingIfPresent();
-    const a = t.split('?');
-    let o = new URLSearchParams();
-    a.length > 1 && (o = new URLSearchParams(a[1])),
-        i && o.set('page', i),
-        0 !== s && o.set('limit', `${s}`);
-    const r = `${a[0]}?${o.toString()}`;
-    $.ajax({ url: r, cache: !1 })
-        .success(function (t) {
-            appendLoadResult(e, n, i, r, t);
-        })
-        .error(function (e) {
-            showError(e), hideLoadingIfPresent();
-        });
-}
-function appendLoadResult(e, t, i, n, s) {
-    if (!1 === additionalLoadAllowed) return void hideLoadingIfPresent();
-    if (activeTab !== e) return void hideLoadingIfPresent();
-    const a = $(`#${e}`),
-        o = document.createElement('div');
-    o.innerHTML = s;
-    const r = $(o);
-    let l = [];
-    if (
-        (t && (l = r.find(`#${t}`)),
-        0 === l.length && (l = r.find('.expandabletable')),
-        0 === l.length)
-    )
-        return void hideLoadingIfPresent();
-    t = l[0].id;
-    const c = [...l[0].rows],
-        d = a.find('tbody')[0],
-        u = document.createElement('template');
-    for (let e = 1; e < c.length; e++) u.content.append(c[e]);
-    d.append(u.content);
-    const h = $('#totalCount')[0];
-    if (h) {
-        const e = d.rows.length - 1;
-        updateLoadingCount(e, h.textContent);
-    }
-    $(o).find('#hasNext')[0] ?
-        loadNextPage(e, n, i + 1, t, 0)
-    :   (hideLoadingIfPresent(),
-        'function' == typeof initSorting && initSorting(t));
-}
-function hideLoadingIfPresent() {
-    'function' == typeof removeLoadInfo && removeLoadInfo();
-}
-function showError(e) {
-    console.error(e);
-}
 function loadMorePrisons(e) {
     hideError(),
         startLoading(),
@@ -8479,7 +8428,6 @@ function startMissionsWorkerIfSupported(e, t, i) {
                 scripts_to_import: i,
                 user_id: user_id,
                 alliance_mission_distance: alliance_mission_distance,
-                leitstelles: leitstelles,
                 leitstelle_latitude: leitstelle_latitude,
                 mission_graphics_lookups: mission_graphics_lookups,
                 mobile_bridge_use: mobile_bridge_use,
@@ -34108,7 +34056,7 @@ Object.values ||
             }
             if (o) return o !== c[0] && c.unshift(o), n[o];
         }
-        function L(e, t, i, n) {
+        function R(e, t, i, n) {
             var s,
                 a,
                 o,
@@ -34162,7 +34110,7 @@ Object.values ||
                     }
             return { state: 'success', data: t };
         }
-        function R() {
+        function L() {
             try {
                 return new e.XMLHttpRequest();
             } catch (e) {}
@@ -34913,12 +34861,12 @@ Object.values ||
                 function i(e, t, i, n) {
                     var s, a, o, r, l, c, d, u, h, p;
                     if (
-                        ((t ? t.ownerDocument || t : W) !== R && L(t),
+                        ((t ? t.ownerDocument || t : W) !== L && R(t),
                         (i = i || []),
                         !e || 'string' != typeof e)
                     )
                         return i;
-                    if (1 !== (r = (t = t || R).nodeType) && 9 !== r) return [];
+                    if (1 !== (r = (t = t || L).nodeType) && 9 !== r) return [];
                     if (O && !n) {
                         if ((s = xe.exec(e)))
                             if ((o = s[1])) {
@@ -35008,7 +34956,7 @@ Object.values ||
                     return (e[$] = !0), e;
                 }
                 function o(e) {
-                    var t = R.createElement('div');
+                    var t = L.createElement('div');
                     try {
                         return !!e(t);
                     } catch (e) {
@@ -35326,7 +35274,7 @@ Object.values ||
                                         )),
                                 k = (V += null == y ? 1 : Math.random() || 0.1);
                             for (
-                                v && ((D = r !== R && r), (z = n));
+                                v && ((D = r !== L && r), (z = n));
                                 null != (u = w[f]);
                                 f++
                             ) {
@@ -35417,8 +35365,8 @@ Object.values ||
                     I,
                     D,
                     j,
-                    L,
                     R,
+                    L,
                     B,
                     O,
                     F,
@@ -35567,13 +35515,13 @@ Object.values ||
                         return !!t && 'HTML' !== t.nodeName;
                     }),
                 (A = i.support = {}),
-                (L = i.setDocument =
+                (R = i.setDocument =
                     function (e) {
                         var t = e ? e.ownerDocument || e : W;
                         return (
-                                t !== R && 9 === t.nodeType && t.documentElement
+                                t !== L && 9 === t.nodeType && t.documentElement
                             ) ?
-                                ((R = t),
+                                ((L = t),
                                 (B = t.documentElement),
                                 (O = !M(t)),
                                 (A.attributes = o(function (e) {
@@ -35841,14 +35789,14 @@ Object.values ||
                                             );
                                         }),
                                 t)
-                            :   R;
+                            :   L;
                     }),
                 (i.matches = function (e, t) {
                     return i(e, null, null, t);
                 }),
                 (i.matchesSelector = function (e, t) {
                     if (
-                        ((e.ownerDocument || e) !== R && L(e),
+                        ((e.ownerDocument || e) !== L && R(e),
                         (t = t.replace(ve, "='$1']")),
                         A.matchesSelector &&
                             O &&
@@ -35864,13 +35812,13 @@ Object.values ||
                             )
                                 return n;
                         } catch (e) {}
-                    return i(t, R, null, [e]).length > 0;
+                    return i(t, L, null, [e]).length > 0;
                 }),
                 (i.contains = function (e, t) {
-                    return (e.ownerDocument || e) !== R && L(e), q(e, t);
+                    return (e.ownerDocument || e) !== L && R(e), q(e, t);
                 }),
                 (i.attr = function (e, i) {
-                    (e.ownerDocument || e) !== R && L(e);
+                    (e.ownerDocument || e) !== L && R(e);
                     var n = E.attrHandle[i.toLowerCase()],
                         s =
                             n && ee.call(E.attrHandle, i.toLowerCase()) ?
@@ -36284,8 +36232,8 @@ Object.values ||
                             },
                             focus: function (e) {
                                 return (
-                                    e === R.activeElement &&
-                                    (!R.hasFocus || R.hasFocus()) &&
+                                    e === L.activeElement &&
+                                    (!L.hasFocus || L.hasFocus()) &&
                                     !!(e.type || e.href || ~e.tabIndex)
                                 );
                             },
@@ -36392,7 +36340,7 @@ Object.values ||
                     (C.prototype = E.filters = E.pseudos),
                     (E.setFilters = new C()),
                     (A.sortStable = $.split('').sort(J).join('') === $),
-                    L(),
+                    R(),
                     [0, 0].sort(J),
                     (A.detectDuplicates = Y),
                     (de.find = i),
@@ -36968,8 +36916,8 @@ Object.values ||
             Ie = /^(?:input|select|textarea|button|object)$/i,
             De = /^(?:a|area)$/i,
             je = /^(?:checked|selected)$/i,
-            Le = de.support.getSetAttribute,
-            Re = de.support.input;
+            Re = de.support.getSetAttribute,
+            Le = de.support.input;
         de.fn.extend({
             attr: function (e, t) {
                 return de.access(this, de.attr, e, t, arguments.length > 1);
@@ -37240,14 +37188,14 @@ Object.values ||
                         for (; (i = a[s++]); )
                             (n = de.propFix[i] || i),
                                 de.expr.match.bool.test(i) ?
-                                    (Re && Le) || !je.test(i) ?
+                                    (Le && Re) || !je.test(i) ?
                                         (e[n] = !1)
                                     :   (e[de.camelCase('default-' + i)] = e[
                                             n
                                         ] =
                                             !1)
                                 :   de.attr(e, i, ''),
-                                e.removeAttribute(Le ? i : n);
+                                e.removeAttribute(Re ? i : n);
                 },
                 attrHooks: {
                     type: {
@@ -37306,8 +37254,8 @@ Object.values ||
                 set: function (e, t, i) {
                     return (
                         !1 === t ? de.removeAttr(e, i)
-                        : (Re && Le) || !je.test(i) ?
-                            e.setAttribute((!Le && de.propFix[i]) || i, i)
+                        : (Le && Re) || !je.test(i) ?
+                            e.setAttribute((!Re && de.propFix[i]) || i, i)
                         :   (e[de.camelCase('default-' + i)] = e[i] = !0),
                         i
                     );
@@ -37316,7 +37264,7 @@ Object.values ||
             de.each(de.expr.match.bool.source.match(/\w+/g), function (e, i) {
                 var n = de.expr.attrHandle[i] || de.find.attr;
                 de.expr.attrHandle[i] =
-                    (Re && Le) || !je.test(i) ?
+                    (Le && Re) || !je.test(i) ?
                         function (e, i, s) {
                             var a = de.expr.attrHandle[i],
                                 o =
@@ -37338,7 +37286,7 @@ Object.values ||
                             );
                         };
             }),
-            (Re && Le) ||
+            (Le && Re) ||
                 (de.attrHooks.value = {
                     set: function (e, t, i) {
                         if (!de.nodeName(e, 'input'))
@@ -37346,7 +37294,7 @@ Object.values ||
                         e.defaultValue = t;
                     },
                 }),
-            Le ||
+            Re ||
                 ((ze = {
                     set: function (e, i, n) {
                         var s = e.getAttributeNode(n);
@@ -39322,8 +39270,8 @@ Object.values ||
             It = /\?/,
             Dt = /#.*$/,
             jt = /([?&])_=[^&]*/,
-            Lt = /^(.*?):[ \t]*([^\r\n]*)\r?$/gm,
-            Rt = /^(?:about|app|app-storage|.+-extension|file|res|widget):$/,
+            Rt = /^(.*?):[ \t]*([^\r\n]*)\r?$/gm,
+            Lt = /^(?:about|app|app-storage|.+-extension|file|res|widget):$/,
             Bt = /^(?:GET|HEAD)$/,
             Ot = /^\/\//,
             Ft = /^([\w.+-]+:)(?:\/\/([^\/?#:]*)(?::(\d+)|)|)/,
@@ -39399,7 +39347,7 @@ Object.values ||
                 ajaxSettings: {
                     url: Pt,
                     type: 'GET',
-                    isLocal: Rt.test(Et[1]),
+                    isLocal: Lt.test(Et[1]),
                     global: !0,
                     processData: !0,
                     async: !0,
@@ -39449,7 +39397,7 @@ Object.values ||
                             (k.readyState = e > 0 ? 4 : 0),
                             (a = (e >= 200 && e < 300) || 304 === e),
                             n && (b = j(h, k, n)),
-                            (b = L(h, b, k, a)),
+                            (b = R(h, b, k, a)),
                             a ?
                                 (h.ifModified &&
                                     ((w =
@@ -39511,7 +39459,7 @@ Object.values ||
                                 var t;
                                 if (2 === y) {
                                     if (!u)
-                                        for (u = {}; (t = Lt.exec(r)); )
+                                        for (u = {}; (t = Rt.exec(r)); )
                                             u[t[1].toLowerCase()] = t[2];
                                     t = u[e.toLowerCase()];
                                 }
@@ -39775,9 +39723,9 @@ Object.values ||
         (de.ajaxSettings.xhr =
             e.ActiveXObject ?
                 function () {
-                    return (!this.isLocal && R()) || B();
+                    return (!this.isLocal && L()) || B();
                 }
-            :   R),
+            :   L),
             (Gt = de.ajaxSettings.xhr()),
             (de.support.cors = !!Gt && 'withCredentials' in Gt),
             (Gt = de.support.ajax = !!Gt) &&
@@ -49068,8 +49016,8 @@ Object.values ||
                         I,
                         D,
                         j,
-                        L,
                         R,
+                        L,
                         B,
                         O,
                         F,
@@ -49318,13 +49266,13 @@ Object.values ||
                                     j = this._daylightSavingAdjust(
                                         new Date(ee, X, 1 - M)
                                     ),
-                                    L = 0;
-                                L < D;
-                                L++
+                                    R = 0;
+                                R < D;
+                                R++
                             ) {
                                 for (
                                     A += '<tr>',
-                                        R =
+                                        L =
                                             u ?
                                                 "<td class='ui-datepicker-week-col'>" +
                                                 this._get(
@@ -49349,7 +49297,7 @@ Object.values ||
                                             !B[0] ||
                                             (J && j < J) ||
                                             (Q && j > Q)),
-                                        (R +=
+                                        (L +=
                                             "<td class='" +
                                             ((w + d + 6) % 7 >= 5 ?
                                                 ' ui-datepicker-week-end'
@@ -49416,7 +49364,7 @@ Object.values ||
                                             '</td>'),
                                         j.setDate(j.getDate() + 1),
                                         (j = this._daylightSavingAdjust(j));
-                                A += R + '</tr>';
+                                A += L + '</tr>';
                             }
                             ++X > 11 && ((X = 0), ee++),
                                 (x += A +=
@@ -60355,7 +60303,7 @@ Object.values ||
                 De(e)
             );
         }
-        function Le(e, t) {
+        function Re(e, t) {
             var i, n, s, a, o, r;
             if (!e || 0 === e.length) throw new Error('latlngs not passed');
             De(e) ||
@@ -60396,7 +60344,7 @@ Object.values ||
                     }
             return A([(c = t.unproject(w(r))).lat + l.lat, c.lng + l.lng]);
         }
-        function Re(e, t) {
+        function Le(e, t) {
             var i,
                 n,
                 s,
@@ -60428,7 +60376,7 @@ Object.values ||
                     );
                 case 'GeometryCollection':
                     for (s = 0, a = o.geometries.length; s < a; s++) {
-                        var u = Re(
+                        var u = Le(
                             {
                                 geometry: o.geometries[s],
                                 type: 'Feature',
@@ -60441,7 +60389,7 @@ Object.values ||
                     return new ki(l);
                 case 'FeatureCollection':
                     for (s = 0, a = o.features.length; s < a; s++) {
-                        var h = Re(o.features[s], t);
+                        var h = Le(o.features[s], t);
                         h && l.push(h);
                     }
                     return new ki(l);
@@ -61271,8 +61219,8 @@ Object.values ||
             jt =
                 ((gt = 'MozPerspective' in gt),
                 !window.L_DISABLE_3D && (It || Dt || gt) && !Pt && !Et),
-            Lt = (Ki = 'undefined' != typeof orientation || I('mobile')) && wt,
-            Rt = Ki && Dt,
+            Rt = (Ki = 'undefined' != typeof orientation || I('mobile')) && wt,
+            Lt = Ki && Dt,
             Bt = !window.PointerEvent && window.MSPointerEvent,
             Ot = !(!window.PointerEvent && !Bt),
             Ft = 'ontouchstart' in window || !!window.TouchEvent,
@@ -61327,8 +61275,8 @@ Object.values ||
                 gecko3d: gt,
                 any3d: jt,
                 mobile: Ki,
-                mobileWebkit: Lt,
-                mobileWebkit3d: Rt,
+                mobileWebkit: Rt,
+                mobileWebkit3d: Lt,
                 msPointer: Bt,
                 pointer: Ot,
                 touch: Nt,
@@ -63597,7 +63545,7 @@ Object.values ||
                     _sqClosestPointOnSegment: Ie,
                     isFlat: De,
                     _flat: je,
-                    polylineCenter: Le,
+                    polylineCenter: Re,
                 }),
                 (Tt = {
                     project: function (e) {
@@ -64605,7 +64553,7 @@ Object.values ||
                 },
                 getCenter: function () {
                     if (this._map)
-                        return Le(this._defaultShape(), this._map.options.crs);
+                        return Re(this._defaultShape(), this._map.options.crs);
                     throw new Error(
                         'Must add layer to map before using getCenter()'
                     );
@@ -64842,7 +64790,7 @@ Object.values ||
                     }
                     var a,
                         o = this.options;
-                    return (o.filter && !o.filter(e)) || !(a = Re(e, o)) ?
+                    return (o.filter && !o.filter(e)) || !(a = Le(e, o)) ?
                             this
                         :   ((a.feature = $e(e)),
                             (a.defaultOptions = a.options),
@@ -65131,7 +65079,7 @@ Object.values ||
                     }
                 },
             }),
-            Li = Di.extend({
+            Ri = Di.extend({
                 _initImage: function () {
                     var e = (this._image = this._url);
                     Z(e, 'leaflet-image-layer'),
@@ -65141,7 +65089,7 @@ Object.values ||
                         (e.onmousemove = r);
                 },
             }),
-            Ri = Pt.extend({
+            Li = Pt.extend({
                 options: {
                     interactive: !1,
                     offset: [0, 0],
@@ -65337,7 +65285,7 @@ Object.values ||
                         );
                     },
                 }),
-                Ri.extend({
+                Li.extend({
                     options: {
                         pane: 'popupPane',
                         offset: [0, 7],
@@ -65364,11 +65312,11 @@ Object.values ||
                                 e._popup.options.autoClose &&
                                 e.removeLayer(e._popup),
                             (e._popup = this),
-                            Ri.prototype.openOn.call(this, e)
+                            Li.prototype.openOn.call(this, e)
                         );
                     },
                     onAdd: function (e) {
-                        Ri.prototype.onAdd.call(this, e),
+                        Li.prototype.onAdd.call(this, e),
                             e.fire('popupopen', { popup: this }),
                             this._source &&
                                 (this._source.fire(
@@ -65380,7 +65328,7 @@ Object.values ||
                                     this._source.on('preclick', me));
                     },
                     onRemove: function (e) {
-                        Ri.prototype.onRemove.call(this, e),
+                        Li.prototype.onRemove.call(this, e),
                             e.fire('popupclose', { popup: this }),
                             this._source &&
                                 (this._source.fire(
@@ -65392,7 +65340,7 @@ Object.values ||
                                     this._source.off('preclick', me));
                     },
                     getEvents: function () {
-                        var e = Ri.prototype.getEvents.call(this);
+                        var e = Li.prototype.getEvents.call(this);
                         return (
                             (void 0 !== this.options.closeOnClick ?
                                 this.options.closeOnClick
@@ -65623,7 +65571,7 @@ Object.values ||
                         13 === e.originalEvent.keyCode && this._openPopup(e);
                     },
                 }),
-                Ri.extend({
+                Li.extend({
                     options: {
                         pane: 'tooltipPane',
                         offset: [0, 0],
@@ -65633,7 +65581,7 @@ Object.values ||
                         opacity: 0.9,
                     },
                     onAdd: function (e) {
-                        Ri.prototype.onAdd.call(this, e),
+                        Li.prototype.onAdd.call(this, e),
                             this.setOpacity(this.options.opacity),
                             e.fire('tooltipopen', { tooltip: this }),
                             this._source &&
@@ -65645,7 +65593,7 @@ Object.values ||
                                 ));
                     },
                     onRemove: function (e) {
-                        Ri.prototype.onRemove.call(this, e),
+                        Li.prototype.onRemove.call(this, e),
                             e.fire('tooltipclose', { tooltip: this }),
                             this._source &&
                                 (this.removeEventParent(this._source),
@@ -65656,7 +65604,7 @@ Object.values ||
                                 ));
                     },
                     getEvents: function () {
-                        var e = Ri.prototype.getEvents.call(this);
+                        var e = Li.prototype.getEvents.call(this);
                         return (
                             this.options.permanent || (e.preclick = this.close),
                             e
@@ -67384,7 +67332,7 @@ Object.values ||
         });
         (Gi.create = Ui),
             (Gi.pointsToPath = M),
-            (Ii.geometryToLayer = Re),
+            (Ii.geometryToLayer = Le),
             (Ii.coordsToLatLng = Oe),
             (Ii.coordsToLatLngs = Fe),
             (Ii.latLngToCoords = Ne),
@@ -67711,7 +67659,7 @@ Object.values ||
                     },
                 })),
             Yi =
-                ((Lt =
+                ((Rt =
                     (hi.addInitHook('addHandler', 'dragging', Ki),
                     hi.mergeOptions({ keyboard: !0, keyboardPanDelta: 80 }),
                     yt.extend({
@@ -67863,8 +67811,8 @@ Object.values ||
                             }
                         },
                     }))),
-                (Rt =
-                    (hi.addInitHook('addHandler', 'keyboard', Lt),
+                (Lt =
+                    (hi.addInitHook('addHandler', 'keyboard', Rt),
                     hi.mergeOptions({
                         scrollWheelZoom: !0,
                         wheelDebounceTime: 40,
@@ -67940,7 +67888,7 @@ Object.values ||
                         },
                     }))),
                 (Bt =
-                    (hi.addInitHook('addHandler', 'scrollWheelZoom', Rt),
+                    (hi.addInitHook('addHandler', 'scrollWheelZoom', Lt),
                     hi.mergeOptions({
                         tapHold: Zt.touchNative && Zt.safari && Zt.mobile,
                         tapTolerance: 15,
@@ -68207,8 +68155,8 @@ Object.values ||
                 (hi.BoxZoom = gt),
                 (hi.DoubleClickZoom = jt),
                 (hi.Drag = Ki),
-                (hi.Keyboard = Lt),
-                (hi.ScrollWheelZoom = Rt),
+                (hi.Keyboard = Rt),
+                (hi.ScrollWheelZoom = Lt),
                 (hi.TapHold = Bt),
                 (hi.TouchZoom = Ot),
                 (e.Bounds = k),
@@ -68220,7 +68168,7 @@ Object.values ||
                 (e.Class = b),
                 (e.Control = pi),
                 (e.DivIcon = Fi),
-                (e.DivOverlay = Ri),
+                (e.DivOverlay = Li),
                 (e.DomEvent = bt),
                 (e.DomUtil = vt),
                 (e.Draggable = bi),
@@ -68250,7 +68198,7 @@ Object.values ||
                 (e.Rectangle = Zi),
                 (e.Renderer = $i),
                 (e.SVG = Gi),
-                (e.SVGOverlay = Li),
+                (e.SVGOverlay = Ri),
                 (e.TileLayer = Hi),
                 (e.Tooltip = Oi),
                 (e.Transformation = z),
@@ -68312,7 +68260,7 @@ Object.values ||
                 (e.stamp = s),
                 (e.svg = Ge),
                 (e.svgOverlay = function (e, t, i) {
-                    return new Li(e, t, i);
+                    return new Ri(e, t, i);
                 }),
                 (e.tileLayer = Ve),
                 (e.tooltip = function (e, t) {
@@ -76032,17 +75980,17 @@ if (
         function j(e) {
             return !!e.getClientRects().length;
         }
-        function L(e) {
+        function R(e) {
             return e.replace(/^-ms-/, 'ms-').replace(/-(\w)/g, function (e, t) {
                 return t.toUpperCase();
             });
         }
-        function R(e, t, i, n, s) {
+        function L(e, t, i, n, s) {
             for (e = s ? e.lastChild : e.firstChild; e; ) {
                 var a = s ? e.previousSibling : e.nextSibling;
                 if (
                     (!i && !1 === t(e)) ||
-                    (!n && !1 === R(e, t, i, n, s)) ||
+                    (!n && !1 === L(e, t, i, n, s)) ||
                     (i && !1 === t(e))
                 )
                     return !1;
@@ -76050,7 +75998,7 @@ if (
             }
         }
         function B(e, t, i, n) {
-            R(e, t, i, n, !0);
+            L(e, t, i, n, !0);
         }
         function O(e, t) {
             var i = (t = t || document).createDocumentFragment(),
@@ -76103,7 +76051,7 @@ if (
                 for (; q(e.parentNode, !0); ) e = e.parentNode;
                 return e;
             };
-            R(e, function (e) {
+            L(e, function (e) {
                 var i = 'ul,ol',
                     n = !q(e, !0);
                 if (n && q(e.parentNode, !0)) {
@@ -76196,7 +76144,7 @@ if (
                 n,
                 s = e.style;
             if (
-                (fe[t] || (fe[t] = L(t)),
+                (fe[t] || (fe[t] = R(t)),
                 (n = s[(t = fe[t])]),
                 'textAlign' === t)
             ) {
@@ -76682,7 +76630,7 @@ if (
         }
         function re(e, t) {
             var i;
-            R(
+            L(
                 e,
                 function (e) {
                     q(e, !0) ?
@@ -76702,8 +76650,8 @@ if (
                 x,
                 T,
                 S,
-                L,
                 R,
+                L,
                 O,
                 $,
                 V,
@@ -76776,7 +76724,7 @@ if (
                             e
                         ),
                         y(o, 'z-index', kt.zIndex),
-                        Re && z(o, 'ie ie' + Re),
+                        Le && z(o, 'ie ie' + Le),
                         (ce = e.required),
                         (e.required = !1);
                     var t = le.formats[kt.format];
@@ -76797,7 +76745,7 @@ if (
                             Y.call('ready'),
                             'onReady' in i && i.onReady.call(_t);
                     };
-                    p(je, 'load', n), 'complete' === Le.readyState && n();
+                    p(je, 'load', n), 'complete' === Re.readyState && n();
                 }),
                 (Ie = function () {
                     var e = kt.plugins;
@@ -76814,21 +76762,21 @@ if (
                         U && U.dateFormat && (kt.dateFormat = U.dateFormat);
                 }),
                 (Me = function () {
-                    (L = r('textarea')),
+                    (R = r('textarea')),
                         (f = r('iframe', {
                             frameborder: 0,
                             allowfullscreen: !0,
                         })),
                         kt.startInSourceMode ?
                             (z(o, 'sourceMode'), g(f))
-                        :   (z(o, 'wysiwygMode'), g(L)),
+                        :   (z(o, 'wysiwygMode'), g(R)),
                         kt.spellcheck || _(o, 'spellcheck', 'false'),
                         'https:' === je.location.protocol &&
                             _(f, 'src', 'javascript:false'),
                         u(o, f),
-                        u(o, L),
+                        u(o, R),
                         _t.dimensions(kt.width || M(e), kt.height || I(e));
-                    var t = Re ? 'ie ie' + Re : '';
+                    var t = Le ? 'ie ie' + Le : '';
                     (t += Ce ? ' ios' : ''),
                         (S = f.contentDocument).open(),
                         S.write(
@@ -76844,21 +76792,21 @@ if (
                         (T = S.body),
                         (x = f.contentWindow),
                         _t.readOnly(!!kt.readOnly),
-                        (Ce || Te || Re) &&
-                            (I(T, '100%'), Re || p(T, 'touchend', _t.focus));
+                        (Ce || Te || Le) &&
+                            (I(T, '100%'), Le || p(T, 'touchend', _t.focus));
                     var i = _(e, 'tabindex');
-                    _(L, 'tabindex', i),
+                    _(R, 'tabindex', i),
                         _(f, 'tabindex', i),
                         (Z = new se(x)),
                         g(e),
                         _t.val(e.value);
                     var n = kt.placeholder || _(e, 'placeholder');
-                    n && ((L.placeholder = n), _(T, 'placeholder', n));
+                    n && ((R.placeholder = n), _(T, 'placeholder', n));
                 }),
                 (Ne = function () {
-                    kt.autoUpdate && (p(T, 'blur', pt), p(L, 'blur', pt)),
+                    kt.autoUpdate && (p(T, 'blur', pt), p(R, 'blur', pt)),
                         null === kt.rtl &&
-                            (kt.rtl = 'rtl' === y(L, 'direction')),
+                            (kt.rtl = 'rtl' === y(R, 'direction')),
                         _t.rtl(!!kt.rtl),
                         kt.autoExpand &&
                             (p(T, 'load', mt, be), p(T, 'input keyup', mt)),
@@ -76874,7 +76822,7 @@ if (
                             'onselectionchange' in S ? 'selectionchange' : (
                                 'keyup focus blur contextmenu mouseup touchend click'
                             );
-                    p(Le, 'click', Xe),
+                    p(Re, 'click', Xe),
                         t &&
                             (p(t, 'reset', Ke),
                             p(t, 'submit', _t.updateOriginal, be)),
@@ -76897,11 +76845,11 @@ if (
                         p(T, 'focus', function () {
                             E(T, 'placeholder');
                         }),
-                        p(L, 'blur', ut),
-                        p(L, 'keyup', ht),
-                        p(L, 'keydown', Ue),
-                        p(L, i, Je),
-                        p(L, n, Qe),
+                        p(R, 'blur', ut),
+                        p(R, 'keyup', ht),
+                        p(R, 'keydown', Ue),
+                        p(R, i, Je),
+                        p(R, n, Qe),
                         p(S, 'mousedown', Ye),
                         p(S, s, st),
                         p(S, 'beforedeactivate keyup mouseup', Ee),
@@ -77039,8 +76987,8 @@ if (
                                 ((C = !1),
                                 g(c),
                                 E(o, 'resizing'),
-                                m(Le, d, s),
-                                m(Le, h, a),
+                                m(Re, d, s),
+                                m(Re, h, a),
                                 e.preventDefault());
                         }),
                         ke && ke.create)
@@ -77062,8 +77010,8 @@ if (
                                 (C = !0),
                                 z(o, 'resizing'),
                                 v(c),
-                                p(Le, d, s),
-                                p(Le, h, a),
+                                p(Re, d, s),
+                                p(Re, h, a),
                                 e.preventDefault();
                         });
                 }),
@@ -77091,8 +77039,8 @@ if (
                     if (j(o)) {
                         if (_t.sourceMode())
                             return (
-                                (t = n ? L.value.length : 0),
-                                void L.setSelectionRange(t, t)
+                                (t = n ? R.value.length : 0),
+                                void R.setSelectionRange(t, t)
                             );
                         if ((G(T), n))
                             for (
@@ -77119,18 +77067,18 @@ if (
                 }),
                 (_t.readOnly = function (e) {
                     return 'boolean' != typeof e ?
-                            !L.readonly
+                            !R.readonly
                         :   ((T.contentEditable = !e),
-                            (L.readonly = !e),
+                            (R.readonly = !e),
                             et(e),
                             _t);
                 }),
                 (_t.rtl = function (e) {
                     var t = e ? 'rtl' : 'ltr';
                     return 'boolean' != typeof e ?
-                            'rtl' === _(L, 'dir')
+                            'rtl' === _(R, 'dir')
                         :   (_(T, 'dir', t),
-                            _(L, 'dir', t),
+                            _(R, 'dir', t),
                             E(o, 'rtl'),
                             E(o, 'ltr'),
                             z(o, t),
@@ -77169,8 +77117,8 @@ if (
                     return pe(e) ?
                             A(o, t)
                         :   ((e = !!e) && (fe = je.pageYOffset),
-                            P(Le.documentElement, t, e),
-                            P(Le.body, t, e),
+                            P(Re.documentElement, t, e),
+                            P(Re.body, t, e),
                             P(o, t, e),
                             _t.width(e ? '100%' : kt.width, !1),
                             _t.height(e ? '100%' : kt.height, !1),
@@ -77189,7 +77137,7 @@ if (
                             var i = kt.resizeMinHeight || kt.height || I(e);
                             ue = { min: i, max: kt.resizeMaxHeight || 2 * i };
                         }
-                        var n = Le.createRange();
+                        var n = Re.createRange();
                         n.selectNodeContents(T);
                         var s = n.getBoundingClientRect(),
                             a = S.documentElement.clientHeight - 1,
@@ -77205,13 +77153,13 @@ if (
                             (Z = null),
                             (O = null),
                             (Y = null),
-                            R && d(R),
-                            m(Le, 'click', Xe);
+                            L && d(L),
+                            m(Re, 'click', Xe);
                         var t = e.form;
                         t &&
                             (m(t, 'reset', Ke),
                             m(t, 'submit', _t.updateOriginal)),
-                            d(L),
+                            d(R),
                             d(l),
                             d(o),
                             delete e._sceditor,
@@ -77223,7 +77171,7 @@ if (
                     var l,
                         c = 'sceditor-' + t;
                     _t.closeDropDown(!0),
-                        (R && A(R, c)) ||
+                        (L && A(L, c)) ||
                             (!1 !== s &&
                                 a(
                                     h(i, ':not(input):not(textarea)'),
@@ -77241,31 +77189,31 @@ if (
                                 kt.dropDownCss
                             )),
                             y(
-                                (R = r('div', {
+                                (L = r('div', {
                                     className: 'sceditor-dropdown ' + c,
                                 })),
                                 l
                             ),
-                            u(R, i),
-                            u(o, R),
-                            p(R, 'click focusin', function (e) {
+                            u(L, i),
+                            u(o, L),
+                            p(L, 'click focusin', function (e) {
                                 e.stopPropagation();
                             }),
                             setTimeout(function () {
-                                if (R) {
-                                    var e = h(R, 'input,textarea')[0];
+                                if (L) {
+                                    var e = h(L, 'input,textarea')[0];
                                     e && e.focus();
                                 }
                             }));
                 }),
                 (Xe = function (e) {
                     3 !== e.which &&
-                        R &&
+                        L &&
                         !e.defaultPrevented &&
                         (pt(), _t.closeDropDown());
                 }),
                 (We = function (e) {
-                    var t = Re || Te,
+                    var t = Le || Te,
                         i = T,
                         n = e.clipboardData,
                         s = function (e) {
@@ -77294,7 +77242,7 @@ if (
                     } else if (!ye) {
                         var c = i.scrollTop;
                         for (
-                            Z.saveRange(), ye = Le.createDocumentFragment();
+                            Z.saveRange(), ye = Re.createDocumentFragment();
                             i.firstChild;
 
                         )
@@ -77328,7 +77276,7 @@ if (
                         _t.wysiwygEditorInsertHtml(n.val, null, !0);
                 }),
                 (_t.closeDropDown = function (e) {
-                    R && (d(R), (R = null)), !0 === e && _t.focus();
+                    L && (d(L), (L = null)), !0 === e && _t.focus();
                 }),
                 (_t.wysiwygEditorInsertHtml = function (e, t, i) {
                     var n,
@@ -77363,18 +77311,18 @@ if (
                 (_t.sourceEditorInsertText = function (e, t) {
                     var i,
                         n,
-                        s = L.selectionStart,
-                        a = L.selectionEnd;
-                    (i = L.scrollTop),
-                        L.focus(),
-                        (n = L.value),
+                        s = R.selectionStart,
+                        a = R.selectionEnd;
+                    (i = R.scrollTop),
+                        R.focus(),
+                        (n = R.value),
                         t && (e += n.substring(s, a) + t),
-                        (L.value =
+                        (R.value =
                             n.substring(0, s) + e + n.substring(a, n.length)),
-                        (L.selectionStart = s + e.length - (t ? t.length : 0)),
-                        (L.selectionEnd = L.selectionStart),
-                        (L.scrollTop = i),
-                        L.focus(),
+                        (R.selectionStart = s + e.length - (t ? t.length : 0)),
+                        (R.selectionEnd = R.selectionStart),
+                        (R.scrollTop = i),
+                        R.focus(),
                         dt();
                 }),
                 (_t.getRangeHelper = function () {
@@ -77382,12 +77330,12 @@ if (
                 }),
                 (_t.sourceEditorCaret = function (e) {
                     return (
-                        L.focus(),
+                        R.focus(),
                         e ?
-                            ((L.selectionStart = e.start),
-                            (L.selectionEnd = e.end),
+                            ((R.selectionStart = e.start),
+                            (R.selectionEnd = e.end),
                             this)
-                        :   { start: L.selectionStart, end: L.selectionEnd }
+                        :   { start: R.selectionStart, end: R.selectionEnd }
                     );
                 }),
                 (_t.val = function (e, t) {
@@ -77451,11 +77399,11 @@ if (
                     return f;
                 }),
                 (_t.getSourceEditorValue = function (e) {
-                    var t = L.value;
+                    var t = R.value;
                     return !1 !== e && 'toHtml' in i && (t = i.toHtml(t)), t;
                 }),
                 (_t.setWysiwygEditorValue = function (e) {
-                    e || (e = '<p>' + (Re ? '' : '<br />') + '</p>'),
+                    e || (e = '<p>' + (Le ? '' : '<br />') + '</p>'),
                         (T.innerHTML = e),
                         Ae(),
                         nt(),
@@ -77463,7 +77411,7 @@ if (
                         mt();
                 }),
                 (_t.setSourceEditorValue = function (e) {
-                    (L.value = e), dt();
+                    (R.value = e), dt();
                 }),
                 (_t.updateOriginal = function () {
                     e.value = _t.val();
@@ -77490,7 +77438,7 @@ if (
                             _t.setWysiwygEditorValue(_t.getSourceEditorValue())
                         :   _t.setSourceEditorValue(_t.getWysiwygEditorValue()),
                         (O = null),
-                        b(L),
+                        b(R),
                         b(f),
                         P(o, 'wysiwygMode', e),
                         P(o, 'sourceMode', !e),
@@ -77499,8 +77447,8 @@ if (
                 }),
                 (it = function () {
                     return (
-                        L.focus(),
-                        L.value.substring(L.selectionStart, L.selectionEnd)
+                        R.focus(),
+                        R.value.substring(R.selectionStart, R.selectionEnd)
                     );
                 }),
                 (ze = function (e, t) {
@@ -77520,7 +77468,7 @@ if (
                             ));
                 }),
                 (Ee = function () {
-                    Re && (O = Z.selectedRange());
+                    Le && (O = Z.selectedRange());
                 }),
                 (_t.execCommand = function (e, t) {
                     var i = !1,
@@ -77684,7 +77632,7 @@ if (
                 }),
                 (Qe = function (e) {
                     Y && Y.call(e.type + 'Event', e, _t);
-                    var t = (e.target === L ? 'scesrc' : 'scewys') + e.type;
+                    var t = (e.target === R ? 'scesrc' : 'scewys') + e.type;
                     ft[t] &&
                         ft[t].forEach(function (t) {
                             t.call(_t, e);
@@ -77711,14 +77659,14 @@ if (
                 (_t.blur = function (e, t, i) {
                     return (
                         me(e) ? _t.bind('blur', e, t, i)
-                        : _t.sourceMode() ? L.blur()
+                        : _t.sourceMode() ? R.blur()
                         : T.blur(),
                         _t
                     );
                 }),
                 (_t.focus = function (e, t, i) {
                     if (me(e)) _t.bind('focus', e, t, i);
-                    else if (_t.inSourceMode()) L.focus();
+                    else if (_t.inSourceMode()) R.focus();
                     else {
                         if (h(S, ':focus').length) return;
                         var n,
@@ -78002,7 +77950,7 @@ if (
                                 !S.getElementById('sceditor-start-marker')),
                             V && (clearTimeout(V), (V = !1)),
                             n && e && Z.saveRange(),
-                            (t = i ? L.value : T.innerHTML) !== dt.lastVal &&
+                            (t = i ? R.value : T.innerHTML) !== dt.lastVal &&
                                 ((dt.lastVal = t),
                                 D(o, 'valuechanged', {
                                     rawValue: i ? _t.val() : t,
@@ -78893,9 +78841,9 @@ if (
                 return { node: r || c, offset: d, text: l };
             },
             je = window,
-            Le = document,
-            Re = xe,
-            Be = Re && Re < 11,
+            Re = document,
+            Le = xe,
+            Be = Le && Le < 11,
             Oe = /^image\/(p?jpe?g|gif|png|bmp)$/i;
         (le.locale = {}),
             (le.formats = {}),
@@ -78948,7 +78896,7 @@ if (
                     closest: c,
                     width: M,
                     height: I,
-                    traverse: R,
+                    traverse: L,
                     rTraverse: B,
                     parseHTML: O,
                     hasStyling: F,
@@ -79240,17 +79188,17 @@ if (
         function j(e) {
             return !!e.getClientRects().length;
         }
-        function L(e) {
+        function R(e) {
             return e.replace(/^-ms-/, 'ms-').replace(/-(\w)/g, function (e, t) {
                 return t.toUpperCase();
             });
         }
-        function R(e, t, i, n, s) {
+        function L(e, t, i, n, s) {
             for (e = s ? e.lastChild : e.firstChild; e; ) {
                 var a = s ? e.previousSibling : e.nextSibling;
                 if (
                     (!i && !1 === t(e)) ||
-                    (!n && !1 === R(e, t, i, n, s)) ||
+                    (!n && !1 === L(e, t, i, n, s)) ||
                     (i && !1 === t(e))
                 )
                     return !1;
@@ -79258,7 +79206,7 @@ if (
             }
         }
         function B(e, t, i, n) {
-            R(e, t, i, n, !0);
+            L(e, t, i, n, !0);
         }
         function O(e, t) {
             var i = (t = t || document).createDocumentFragment(),
@@ -79311,7 +79259,7 @@ if (
                 for (; q(e.parentNode, !0); ) e = e.parentNode;
                 return e;
             };
-            R(e, function (e) {
+            L(e, function (e) {
                 var i = 'ul,ol',
                     n = !q(e, !0);
                 if (n && q(e.parentNode, !0)) {
@@ -79404,7 +79352,7 @@ if (
                 n,
                 s = e.style;
             if (
-                (fe[t] || (fe[t] = L(t)),
+                (fe[t] || (fe[t] = R(t)),
                 (n = s[(t = fe[t])]),
                 'textAlign' === t)
             ) {
@@ -79890,7 +79838,7 @@ if (
         }
         function re(e, t) {
             var i;
-            R(
+            L(
                 e,
                 function (e) {
                     q(e, !0) ?
@@ -79910,8 +79858,8 @@ if (
                 x,
                 T,
                 S,
-                L,
                 R,
+                L,
                 O,
                 $,
                 V,
@@ -79984,7 +79932,7 @@ if (
                             e
                         ),
                         y(o, 'z-index', kt.zIndex),
-                        Re && z(o, 'ie ie' + Re),
+                        Le && z(o, 'ie ie' + Le),
                         (ce = e.required),
                         (e.required = !1);
                     var t = le.formats[kt.format];
@@ -80005,7 +79953,7 @@ if (
                             Y.call('ready'),
                             'onReady' in i && i.onReady.call(_t);
                     };
-                    p(je, 'load', n), 'complete' === Le.readyState && n();
+                    p(je, 'load', n), 'complete' === Re.readyState && n();
                 }),
                 (Ie = function () {
                     var e = kt.plugins;
@@ -80022,21 +79970,21 @@ if (
                         U && U.dateFormat && (kt.dateFormat = U.dateFormat);
                 }),
                 (Me = function () {
-                    (L = r('textarea')),
+                    (R = r('textarea')),
                         (f = r('iframe', {
                             frameborder: 0,
                             allowfullscreen: !0,
                         })),
                         kt.startInSourceMode ?
                             (z(o, 'sourceMode'), g(f))
-                        :   (z(o, 'wysiwygMode'), g(L)),
+                        :   (z(o, 'wysiwygMode'), g(R)),
                         kt.spellcheck || _(o, 'spellcheck', 'false'),
                         'https:' === je.location.protocol &&
                             _(f, 'src', 'javascript:false'),
                         u(o, f),
-                        u(o, L),
+                        u(o, R),
                         _t.dimensions(kt.width || M(e), kt.height || I(e));
-                    var t = Re ? 'ie ie' + Re : '';
+                    var t = Le ? 'ie ie' + Le : '';
                     (t += Ce ? ' ios' : ''),
                         (S = f.contentDocument).open(),
                         S.write(
@@ -80052,21 +80000,21 @@ if (
                         (T = S.body),
                         (x = f.contentWindow),
                         _t.readOnly(!!kt.readOnly),
-                        (Ce || Te || Re) &&
-                            (I(T, '100%'), Re || p(T, 'touchend', _t.focus));
+                        (Ce || Te || Le) &&
+                            (I(T, '100%'), Le || p(T, 'touchend', _t.focus));
                     var i = _(e, 'tabindex');
-                    _(L, 'tabindex', i),
+                    _(R, 'tabindex', i),
                         _(f, 'tabindex', i),
                         (Z = new se(x)),
                         g(e),
                         _t.val(e.value);
                     var n = kt.placeholder || _(e, 'placeholder');
-                    n && ((L.placeholder = n), _(T, 'placeholder', n));
+                    n && ((R.placeholder = n), _(T, 'placeholder', n));
                 }),
                 (Ne = function () {
-                    kt.autoUpdate && (p(T, 'blur', pt), p(L, 'blur', pt)),
+                    kt.autoUpdate && (p(T, 'blur', pt), p(R, 'blur', pt)),
                         null === kt.rtl &&
-                            (kt.rtl = 'rtl' === y(L, 'direction')),
+                            (kt.rtl = 'rtl' === y(R, 'direction')),
                         _t.rtl(!!kt.rtl),
                         kt.autoExpand &&
                             (p(T, 'load', mt, be), p(T, 'input keyup', mt)),
@@ -80082,7 +80030,7 @@ if (
                             'onselectionchange' in S ? 'selectionchange' : (
                                 'keyup focus blur contextmenu mouseup touchend click'
                             );
-                    p(Le, 'click', Xe),
+                    p(Re, 'click', Xe),
                         t &&
                             (p(t, 'reset', Ke),
                             p(t, 'submit', _t.updateOriginal, be)),
@@ -80105,11 +80053,11 @@ if (
                         p(T, 'focus', function () {
                             E(T, 'placeholder');
                         }),
-                        p(L, 'blur', ut),
-                        p(L, 'keyup', ht),
-                        p(L, 'keydown', Ue),
-                        p(L, i, Je),
-                        p(L, n, Qe),
+                        p(R, 'blur', ut),
+                        p(R, 'keyup', ht),
+                        p(R, 'keydown', Ue),
+                        p(R, i, Je),
+                        p(R, n, Qe),
                         p(S, 'mousedown', Ye),
                         p(S, s, st),
                         p(S, 'beforedeactivate keyup mouseup', Ee),
@@ -80247,8 +80195,8 @@ if (
                                 ((C = !1),
                                 g(c),
                                 E(o, 'resizing'),
-                                m(Le, d, s),
-                                m(Le, h, a),
+                                m(Re, d, s),
+                                m(Re, h, a),
                                 e.preventDefault());
                         }),
                         ke && ke.create)
@@ -80270,8 +80218,8 @@ if (
                                 (C = !0),
                                 z(o, 'resizing'),
                                 v(c),
-                                p(Le, d, s),
-                                p(Le, h, a),
+                                p(Re, d, s),
+                                p(Re, h, a),
                                 e.preventDefault();
                         });
                 }),
@@ -80299,8 +80247,8 @@ if (
                     if (j(o)) {
                         if (_t.sourceMode())
                             return (
-                                (t = n ? L.value.length : 0),
-                                void L.setSelectionRange(t, t)
+                                (t = n ? R.value.length : 0),
+                                void R.setSelectionRange(t, t)
                             );
                         if ((G(T), n))
                             for (
@@ -80327,18 +80275,18 @@ if (
                 }),
                 (_t.readOnly = function (e) {
                     return 'boolean' != typeof e ?
-                            !L.readonly
+                            !R.readonly
                         :   ((T.contentEditable = !e),
-                            (L.readonly = !e),
+                            (R.readonly = !e),
                             et(e),
                             _t);
                 }),
                 (_t.rtl = function (e) {
                     var t = e ? 'rtl' : 'ltr';
                     return 'boolean' != typeof e ?
-                            'rtl' === _(L, 'dir')
+                            'rtl' === _(R, 'dir')
                         :   (_(T, 'dir', t),
-                            _(L, 'dir', t),
+                            _(R, 'dir', t),
                             E(o, 'rtl'),
                             E(o, 'ltr'),
                             z(o, t),
@@ -80377,8 +80325,8 @@ if (
                     return pe(e) ?
                             A(o, t)
                         :   ((e = !!e) && (fe = je.pageYOffset),
-                            P(Le.documentElement, t, e),
-                            P(Le.body, t, e),
+                            P(Re.documentElement, t, e),
+                            P(Re.body, t, e),
                             P(o, t, e),
                             _t.width(e ? '100%' : kt.width, !1),
                             _t.height(e ? '100%' : kt.height, !1),
@@ -80397,7 +80345,7 @@ if (
                             var i = kt.resizeMinHeight || kt.height || I(e);
                             ue = { min: i, max: kt.resizeMaxHeight || 2 * i };
                         }
-                        var n = Le.createRange();
+                        var n = Re.createRange();
                         n.selectNodeContents(T);
                         var s = n.getBoundingClientRect(),
                             a = S.documentElement.clientHeight - 1,
@@ -80413,13 +80361,13 @@ if (
                             (Z = null),
                             (O = null),
                             (Y = null),
-                            R && d(R),
-                            m(Le, 'click', Xe);
+                            L && d(L),
+                            m(Re, 'click', Xe);
                         var t = e.form;
                         t &&
                             (m(t, 'reset', Ke),
                             m(t, 'submit', _t.updateOriginal)),
-                            d(L),
+                            d(R),
                             d(l),
                             d(o),
                             delete e._sceditor,
@@ -80431,7 +80379,7 @@ if (
                     var l,
                         c = 'sceditor-' + t;
                     _t.closeDropDown(!0),
-                        (R && A(R, c)) ||
+                        (L && A(L, c)) ||
                             (!1 !== s &&
                                 a(
                                     h(i, ':not(input):not(textarea)'),
@@ -80449,31 +80397,31 @@ if (
                                 kt.dropDownCss
                             )),
                             y(
-                                (R = r('div', {
+                                (L = r('div', {
                                     className: 'sceditor-dropdown ' + c,
                                 })),
                                 l
                             ),
-                            u(R, i),
-                            u(o, R),
-                            p(R, 'click focusin', function (e) {
+                            u(L, i),
+                            u(o, L),
+                            p(L, 'click focusin', function (e) {
                                 e.stopPropagation();
                             }),
                             setTimeout(function () {
-                                if (R) {
-                                    var e = h(R, 'input,textarea')[0];
+                                if (L) {
+                                    var e = h(L, 'input,textarea')[0];
                                     e && e.focus();
                                 }
                             }));
                 }),
                 (Xe = function (e) {
                     3 !== e.which &&
-                        R &&
+                        L &&
                         !e.defaultPrevented &&
                         (pt(), _t.closeDropDown());
                 }),
                 (We = function (e) {
-                    var t = Re || Te,
+                    var t = Le || Te,
                         i = T,
                         n = e.clipboardData,
                         s = function (e) {
@@ -80502,7 +80450,7 @@ if (
                     } else if (!ye) {
                         var c = i.scrollTop;
                         for (
-                            Z.saveRange(), ye = Le.createDocumentFragment();
+                            Z.saveRange(), ye = Re.createDocumentFragment();
                             i.firstChild;
 
                         )
@@ -80536,7 +80484,7 @@ if (
                         _t.wysiwygEditorInsertHtml(n.val, null, !0);
                 }),
                 (_t.closeDropDown = function (e) {
-                    R && (d(R), (R = null)), !0 === e && _t.focus();
+                    L && (d(L), (L = null)), !0 === e && _t.focus();
                 }),
                 (_t.wysiwygEditorInsertHtml = function (e, t, i) {
                     var n,
@@ -80571,18 +80519,18 @@ if (
                 (_t.sourceEditorInsertText = function (e, t) {
                     var i,
                         n,
-                        s = L.selectionStart,
-                        a = L.selectionEnd;
-                    (i = L.scrollTop),
-                        L.focus(),
-                        (n = L.value),
+                        s = R.selectionStart,
+                        a = R.selectionEnd;
+                    (i = R.scrollTop),
+                        R.focus(),
+                        (n = R.value),
                         t && (e += n.substring(s, a) + t),
-                        (L.value =
+                        (R.value =
                             n.substring(0, s) + e + n.substring(a, n.length)),
-                        (L.selectionStart = s + e.length - (t ? t.length : 0)),
-                        (L.selectionEnd = L.selectionStart),
-                        (L.scrollTop = i),
-                        L.focus(),
+                        (R.selectionStart = s + e.length - (t ? t.length : 0)),
+                        (R.selectionEnd = R.selectionStart),
+                        (R.scrollTop = i),
+                        R.focus(),
                         dt();
                 }),
                 (_t.getRangeHelper = function () {
@@ -80590,12 +80538,12 @@ if (
                 }),
                 (_t.sourceEditorCaret = function (e) {
                     return (
-                        L.focus(),
+                        R.focus(),
                         e ?
-                            ((L.selectionStart = e.start),
-                            (L.selectionEnd = e.end),
+                            ((R.selectionStart = e.start),
+                            (R.selectionEnd = e.end),
                             this)
-                        :   { start: L.selectionStart, end: L.selectionEnd }
+                        :   { start: R.selectionStart, end: R.selectionEnd }
                     );
                 }),
                 (_t.val = function (e, t) {
@@ -80659,11 +80607,11 @@ if (
                     return f;
                 }),
                 (_t.getSourceEditorValue = function (e) {
-                    var t = L.value;
+                    var t = R.value;
                     return !1 !== e && 'toHtml' in i && (t = i.toHtml(t)), t;
                 }),
                 (_t.setWysiwygEditorValue = function (e) {
-                    e || (e = '<p>' + (Re ? '' : '<br />') + '</p>'),
+                    e || (e = '<p>' + (Le ? '' : '<br />') + '</p>'),
                         (T.innerHTML = e),
                         Ae(),
                         nt(),
@@ -80671,7 +80619,7 @@ if (
                         mt();
                 }),
                 (_t.setSourceEditorValue = function (e) {
-                    (L.value = e), dt();
+                    (R.value = e), dt();
                 }),
                 (_t.updateOriginal = function () {
                     e.value = _t.val();
@@ -80698,7 +80646,7 @@ if (
                             _t.setWysiwygEditorValue(_t.getSourceEditorValue())
                         :   _t.setSourceEditorValue(_t.getWysiwygEditorValue()),
                         (O = null),
-                        b(L),
+                        b(R),
                         b(f),
                         P(o, 'wysiwygMode', e),
                         P(o, 'sourceMode', !e),
@@ -80707,8 +80655,8 @@ if (
                 }),
                 (it = function () {
                     return (
-                        L.focus(),
-                        L.value.substring(L.selectionStart, L.selectionEnd)
+                        R.focus(),
+                        R.value.substring(R.selectionStart, R.selectionEnd)
                     );
                 }),
                 (ze = function (e, t) {
@@ -80728,7 +80676,7 @@ if (
                             ));
                 }),
                 (Ee = function () {
-                    Re && (O = Z.selectedRange());
+                    Le && (O = Z.selectedRange());
                 }),
                 (_t.execCommand = function (e, t) {
                     var i = !1,
@@ -80892,7 +80840,7 @@ if (
                 }),
                 (Qe = function (e) {
                     Y && Y.call(e.type + 'Event', e, _t);
-                    var t = (e.target === L ? 'scesrc' : 'scewys') + e.type;
+                    var t = (e.target === R ? 'scesrc' : 'scewys') + e.type;
                     ft[t] &&
                         ft[t].forEach(function (t) {
                             t.call(_t, e);
@@ -80919,14 +80867,14 @@ if (
                 (_t.blur = function (e, t, i) {
                     return (
                         me(e) ? _t.bind('blur', e, t, i)
-                        : _t.sourceMode() ? L.blur()
+                        : _t.sourceMode() ? R.blur()
                         : T.blur(),
                         _t
                     );
                 }),
                 (_t.focus = function (e, t, i) {
                     if (me(e)) _t.bind('focus', e, t, i);
-                    else if (_t.inSourceMode()) L.focus();
+                    else if (_t.inSourceMode()) R.focus();
                     else {
                         if (h(S, ':focus').length) return;
                         var n,
@@ -81210,7 +81158,7 @@ if (
                                 !S.getElementById('sceditor-start-marker')),
                             V && (clearTimeout(V), (V = !1)),
                             n && e && Z.saveRange(),
-                            (t = i ? L.value : T.innerHTML) !== dt.lastVal &&
+                            (t = i ? R.value : T.innerHTML) !== dt.lastVal &&
                                 ((dt.lastVal = t),
                                 D(o, 'valuechanged', {
                                     rawValue: i ? _t.val() : t,
@@ -82101,9 +82049,9 @@ if (
                 return { node: r || c, offset: d, text: l };
             },
             je = window,
-            Le = document,
-            Re = xe,
-            Be = Re && Re < 11,
+            Re = document,
+            Le = xe,
+            Be = Le && Le < 11,
             Oe = /^image\/(p?jpe?g|gif|png|bmp)$/i;
         (le.locale = {}),
             (le.formats = {}),
@@ -82156,7 +82104,7 @@ if (
                     closest: c,
                     width: M,
                     height: I,
-                    traverse: R,
+                    traverse: L,
                     rTraverse: B,
                     parseHTML: O,
                     hasStyling: F,
