diff --git a/application.js b/application.alt.js
index c8262b4..b6df152 100644
--- a/application.js
+++ b/application.alt.js
@@ -7468,7 +7468,6 @@ Object.values ||
             ready_home: 'Available at Station',
             ready_traveling: 'Clear and Available',
             talking_wish: 'Transport Request',
-            transporting_car: 'Transporting Vehicle',
             waiting_for_vehicle: 'Awaiting vehicle to tow',
         },
         intervention_order: {
@@ -8039,8 +8038,6 @@ Object.values ||
         },
         javascript: {
             alarm: 'Dispatch',
-            alliance_event_pay_out_message:
-                'This will be paid out when all the alliance event missions are finished',
             alliance_event_resource: {
                 christmas: {
                     currency: 'Event Tickets',
@@ -8063,8 +8060,6 @@ Object.values ||
             backalarm: 'Cancel',
             coins: 'Coins',
             credits: 'Credits',
-            current_reward_for_you:
-                '<b>Current Reward for you:</b> %{credits} Credits & %{event_currency} Event currency',
             days: 'days',
             few_seconds: 'in few seconds',
             finish_in: 'Finish in:',
@@ -8081,8 +8076,6 @@ Object.values ||
             mission_start_in: 'Starts in:',
             new: 'New',
             not_found_map: "The vehicle hasn't been found on the map",
-            not_qualified:
-                '<b>Not qualified:</b> Participate in a mission to earn the reward ',
             now: 'Now',
             patient: 'Patient',
             patient_untouched: 'Untreated patients',
@@ -8092,7 +8085,6 @@ Object.values ||
             pump_speed_on_site: 'On scene: %{amount} l/min',
             pump_speed_process: 'Water Damage Pump Process',
             pump_speed_selected: 'Selected: %{amount} l/min',
-            qualified: '<b>Qualified:</b> You took part in the event',
             reload: 'Reload',
             sale: 'Sale',
             sale_ended: 'Sale ended',
@@ -8110,8 +8102,6 @@ Object.values ||
             time_left: 'Time left:',
             to_building: 'View Building',
             to_mission: 'View Mission',
-            total_credits_earned: 'Total credits earned:',
-            total_event_currency_earned: 'Total Event currency earned:',
             understand: 'Acknowledge',
             user_not_found: 'The player was not found.',
             vehicle_payload: {
@@ -8857,7 +8847,6 @@ Object.values ||
                 technical_aid: 'Steunpunt Rijkswaterstaat',
                 technical_aid_missions: 'RWS / Signalisatie Inzetten',
                 technical_aid_organization: 'THW',
-                tow_trucks: 'Berger',
                 user_buildings: 'Mijn gebouwen',
                 user_missions: 'Mijn inzetten',
                 water_watch: 'Waterongevallen',
@@ -8946,14 +8935,11 @@ Object.values ||
             ready_home: 'Op post',
             ready_traveling: 'Beschikbaar',
             talking_wish: 'Aanvraag spraakcontact',
-            transporting_car: 'Voertuig in Transport',
             waiting_for_vehicle: 'Wachten op trekkend voertuig.',
         },
         intervention_order: {
             automatic_text_color: 'Automatische tekstkleur',
             back: 'Terug',
-            car_carrier: 'Elke Berger-K',
-            car_carrier_large: 'Elke Berger-G',
             categories: {
                 brush: 'NBB',
                 coastal_rescue: 'Kustwacht',
@@ -9021,8 +9007,6 @@ Object.values ||
                 bike_police: 'DB-Bike',
                 boot: 'WOA of BA-RB',
                 brush_truck: 'Terreinvaardige Tankautospuit',
-                car_carrier: 'Een Berger-K',
-                car_carrier_large: 'Een Berger-G',
                 coastal_boat: 'RB-K of RB-G',
                 division_chief_unit: 'Hoofdofficier van Dienst - Brandweer',
                 elw_airport: 'Airport Fire Officer / On Scene Commander',
@@ -55500,7 +55484,6 @@ const AAO_MULTIPLE_KEYS = [
     'water_amount_tlf_water_carrier',
     'police_horse_count',
     'car_carrier',
-    'car_large',
 ];
 var lastMissionTimeoutID = null,
     hideTimeOut,
@@ -56012,10 +55995,6 @@ $(function () {
                 'railway_fire',
                 I18n.t('intervention_order.vehicles.railway_fire'),
             ],
-            [
-                'car_carrier_large',
-                I18n.t('intervention_order.vehicles.car_carrier_large'),
-            ],
         ]),
         $('#restore_map').click(function () {
             mapViewRestore();
